{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "a2a6bcde",
   "metadata": {
    "slideshow": {
     "slide_type": "skip"
    },
    "tags": [
     "snakemake-job-properties",
     "remove-input"
    ],
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "\n",
    "######## snakemake preamble start (automatically inserted, do not edit) ########\n",
    "library(methods)\n",
    "Snakemake <- setClass(\n",
    "    \"Snakemake\",\n",
    "    slots = c(\n",
    "        input = \"list\",\n",
    "        output = \"list\",\n",
    "        params = \"list\",\n",
    "        wildcards = \"list\",\n",
    "        threads = \"numeric\",\n",
    "        log = \"list\",\n",
    "        resources = \"list\",\n",
    "        config = \"list\",\n",
    "        rule = \"character\",\n",
    "        bench_iteration = \"numeric\",\n",
    "        scriptdir = \"character\",\n",
    "        source = \"function\"\n",
    "    )\n",
    ")\n",
    "snakemake <- Snakemake(\n",
    "    input = list(),\n",
    "    output = list(),\n",
    "    params = list(),\n",
    "    wildcards = list(),\n",
    "    threads = 1,\n",
    "    log = list( 'rds' = 'out/ipynb/compute_unsupervised_classification/pca.rds'),\n",
    "    resources = list('tmpdir', \"tmpdir\" = '/tmp'),\n",
    "    config = list(),\n",
    "    rule = 'ipynb',\n",
    "    bench_iteration = as.numeric(NA),\n",
    "    scriptdir = '/mnt/thymus/synoSalva/illumina_sequencing_data/mw/mw-oncodiag/src/snakemake/rules/../../ipynb',\n",
    "    source = function(...){\n",
    "        wd <- getwd()\n",
    "        setwd(snakemake@scriptdir)\n",
    "        source(...)\n",
    "        setwd(wd)\n",
    "    }\n",
    ")\n",
    "# setwd('/mnt/thymus/synoSalva/illumina_sequencing_data/mw/mw-oncodiag');\n",
    "setwd('/srv/shiny-server/mw-oncodiag');\n",
    "\n",
    "######## snakemake preamble end #########\n",
    "# context <- \"CpG\"\n",
    "context <- \"CHH\"\n",
    "# context <- \"CHG\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "8e449f62",
   "metadata": {
    "slideshow": {
     "slide_type": "skip"
    },
    "tags": [
     "remove-input"
    ],
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# start coding here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "7690591a-7450-4f4f-9f1d-1b9540939f47",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/markdown": []
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "IRdisplay::display_markdown(\n",
    "    sprintf(\n",
    "\"_Compute unsupervised classification %s_\n",
    "\n",
    "__Oncodiag project__\n",
    "\n",
    "[Guillaume Charbonnier](mailto:gc.bioinfo@gmail.com)\n",
    "\n",
    "%s\",\n",
    "        snakemake@wildcards$design,\n",
    "        format(Sys.Date(), \"%Y-%m-%d\")\n",
    "    )\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "082c2ea6-9d05-48c3-9cf2-ee88852b2844",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    },
    "tags": []
   },
   "source": [
    "# Abstract\n",
    "\n",
    "The aim of this analysis is to assess the main components of variance in the dataset."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b43c03ed-8ab7-482b-8797-bf9c310f6db1",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    },
    "tags": []
   },
   "source": [
    "# Methods\n",
    "\n",
    "## Load depencies\n",
    "\n",
    "### Packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "8a4fe98c-a611-4d31-8303-f02876ccc4cd",
   "metadata": {
    "tags": [],
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'getOption(\"repos\")' replaces Bioconductor standard repositories, see\n",
      "'help(\"repositories\", package = \"BiocManager\")' for details.\n",
      "Replacement repositories:\n",
      "    CRAN: https://cran.r-project.org\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Bioconductor version 3.18 (BiocManager 1.30.22), R 4.3.2 (2023-10-31)\n",
      "\n",
      "Warning message:\n",
      "“package(s) not installed when version(s) same as or greater than current; use\n",
      "  `force = TRUE` to re-install: 'data.table' 'openxlsx' 'DSS' 'bsseq'\n",
      "  'ChIPpeakAnno' 'irlba' 'org.Hs.eg.db' 'TxDb.Hsapiens.UCSC.hg19.knownGene'\n",
      "  'ggpubr' 'tidyr'”\n",
      "Loading required package: Biobase\n",
      "\n",
      "Loading required package: BiocGenerics\n",
      "\n",
      "\n",
      "Attaching package: ‘BiocGenerics’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    IQR, mad, sd, var, xtabs\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    anyDuplicated, aperm, append, as.data.frame, basename, cbind,\n",
      "    colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,\n",
      "    get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,\n",
      "    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,\n",
      "    Position, rank, rbind, Reduce, rownames, sapply, setdiff, sort,\n",
      "    table, tapply, union, unique, unsplit, which.max, which.min\n",
      "\n",
      "\n",
      "Welcome to Bioconductor\n",
      "\n",
      "    Vignettes contain introductory material; view with\n",
      "    'browseVignettes()'. To cite Bioconductor, see\n",
      "    'citation(\"Biobase\")', and for packages 'citation(\"pkgname\")'.\n",
      "\n",
      "\n",
      "Loading required package: BiocParallel\n",
      "\n",
      "Loading required package: bsseq\n",
      "\n",
      "Loading required package: GenomicRanges\n",
      "\n",
      "Loading required package: stats4\n",
      "\n",
      "Loading required package: S4Vectors\n",
      "\n",
      "\n",
      "Attaching package: ‘S4Vectors’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:data.table’:\n",
      "\n",
      "    first, second\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:utils’:\n",
      "\n",
      "    findMatches\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    expand.grid, I, unname\n",
      "\n",
      "\n",
      "Loading required package: IRanges\n",
      "\n",
      "\n",
      "Attaching package: ‘IRanges’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:data.table’:\n",
      "\n",
      "    shift\n",
      "\n",
      "\n",
      "Loading required package: GenomeInfoDb\n",
      "\n",
      "Loading required package: SummarizedExperiment\n",
      "\n",
      "Loading required package: MatrixGenerics\n",
      "\n",
      "Loading required package: matrixStats\n",
      "\n",
      "\n",
      "Attaching package: ‘matrixStats’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:Biobase’:\n",
      "\n",
      "    anyMissing, rowMedians\n",
      "\n",
      "\n",
      "\n",
      "Attaching package: ‘MatrixGenerics’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:matrixStats’:\n",
      "\n",
      "    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,\n",
      "    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,\n",
      "    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,\n",
      "    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,\n",
      "    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,\n",
      "    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,\n",
      "    colWeightedMeans, colWeightedMedians, colWeightedSds,\n",
      "    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,\n",
      "    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,\n",
      "    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,\n",
      "    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,\n",
      "    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,\n",
      "    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,\n",
      "    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,\n",
      "    rowWeightedSds, rowWeightedVars\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:Biobase’:\n",
      "\n",
      "    rowMedians\n",
      "\n",
      "\n",
      "Loading required package: parallel\n",
      "\n",
      "Loading required package: Matrix\n",
      "\n",
      "\n",
      "Attaching package: ‘Matrix’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:S4Vectors’:\n",
      "\n",
      "    expand\n",
      "\n",
      "\n",
      "Loading required package: AnnotationDbi\n",
      "\n",
      "\n",
      "\n",
      "Loading required package: GenomicFeatures\n",
      "\n",
      "Loading required package: ggplot2\n",
      "\n",
      "\n",
      "Attaching package: ‘tidyr’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:Matrix’:\n",
      "\n",
      "    expand, pack, unpack\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:S4Vectors’:\n",
      "\n",
      "    expand\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "if (!require(\"BiocManager\", quietly = TRUE)) {\n",
    "    install.packages(\n",
    "        \"BiocManager\",\n",
    "        quiet = TRUE\n",
    "    )\n",
    "}\n",
    "packages <- c(\n",
    "  \"data.table\",\n",
    "  # \"dplyr\",\n",
    "  \"openxlsx\",\n",
    "  # \"methylSig\",\n",
    "  \"DSS\",\n",
    "  \"bsseq\",\n",
    "  \"ChIPpeakAnno\",\n",
    "  \"irlba\",\n",
    "  # \"ChIPseeker\",\n",
    "  # \"methyAnalysis\",\n",
    "  \"org.Hs.eg.db\",\n",
    "  \"TxDb.Hsapiens.UCSC.hg19.knownGene\",\n",
    "  # \"EnhancedVolcano\",\n",
    "  # \"data.table\",\n",
    "  \"ggpubr\",\n",
    "  # \"ComplexHeatmap\",\n",
    "  \"tidyr\"\n",
    ")\n",
    "BiocManager::install(\n",
    "    packages,\n",
    "    update = FALSE,\n",
    "    quiet = TRUE,\n",
    "    Ncpus = parallel::detectCores()\n",
    ")\n",
    "invisible(\n",
    "    lapply(\n",
    "      packages,\n",
    "      library,\n",
    "      character.only = TRUE\n",
    "    )\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "2518be08-c1f3-4fb4-a1d8-0238a0c0e829",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 4.3.2 (2023-10-31)\n",
       "Platform: x86_64-conda-linux-gnu (64-bit)\n",
       "Running under: Pop!_OS 22.04 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS/LAPACK: /home/gcharbonnier/mambaforge/envs/rkernel_diff_meth_v2/lib/libopenblasp-r0.3.24.so;  LAPACK version 3.11.0\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_DK.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_DK.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_DK.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_DK.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "time zone: Europe/Paris\n",
       "tzcode source: system (glibc)\n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats4    stats     graphics  grDevices utils     datasets \n",
       "[8] methods   base     \n",
       "\n",
       "other attached packages:\n",
       " [1] tidyr_1.3.0                            \n",
       " [2] ggpubr_0.6.0                           \n",
       " [3] ggplot2_3.4.4                          \n",
       " [4] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2\n",
       " [5] GenomicFeatures_1.52.1                 \n",
       " [6] org.Hs.eg.db_3.18.0                    \n",
       " [7] AnnotationDbi_1.64.1                   \n",
       " [8] irlba_2.3.5.1                          \n",
       " [9] Matrix_1.6-1.1                         \n",
       "[10] ChIPpeakAnno_3.36.0                    \n",
       "[11] DSS_2.48.0                             \n",
       "[12] bsseq_1.38.0                           \n",
       "[13] SummarizedExperiment_1.30.2            \n",
       "[14] MatrixGenerics_1.12.2                  \n",
       "[15] matrixStats_1.0.0                      \n",
       "[16] GenomicRanges_1.52.0                   \n",
       "[17] GenomeInfoDb_1.36.1                    \n",
       "[18] IRanges_2.34.1                         \n",
       "[19] S4Vectors_0.38.1                       \n",
       "[20] BiocParallel_1.34.2                    \n",
       "[21] Biobase_2.60.0                         \n",
       "[22] BiocGenerics_0.46.0                    \n",
       "[23] openxlsx_4.2.5.2                       \n",
       "[24] data.table_1.14.8                      \n",
       "[25] BiocManager_1.30.22                    \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       "  [1] jsonlite_1.8.7            magrittr_2.0.3           \n",
       "  [3] BiocIO_1.10.0             zlibbioc_1.46.0          \n",
       "  [5] vctrs_0.6.4               multtest_2.58.0          \n",
       "  [7] memoise_2.0.1             Rsamtools_2.16.0         \n",
       "  [9] DelayedMatrixStats_1.22.1 RCurl_1.98-1.13          \n",
       " [11] base64enc_0.1-3           rstatix_0.7.2            \n",
       " [13] htmltools_0.5.7           S4Arrays_1.0.4           \n",
       " [15] progress_1.2.2            lambda.r_1.2.4           \n",
       " [17] curl_5.1.0                broom_1.0.5              \n",
       " [19] Rhdf5lib_1.22.0           rhdf5_2.44.0             \n",
       " [21] futile.options_1.0.1      cachem_1.0.8             \n",
       " [23] uuid_1.1-1                GenomicAlignments_1.36.0 \n",
       " [25] lifecycle_1.0.3           pkgconfig_2.0.3          \n",
       " [27] R6_2.5.1                  fastmap_1.1.1            \n",
       " [29] GenomeInfoDbData_1.2.10   digest_0.6.33            \n",
       " [31] colorspace_2.1-0          regioneR_1.34.0          \n",
       " [33] RSQLite_2.3.3             filelock_1.0.2           \n",
       " [35] fansi_1.0.5               abind_1.4-5              \n",
       " [37] httr_1.4.7                compiler_4.3.2           \n",
       " [39] withr_2.5.2               bit64_4.0.5              \n",
       " [41] backports_1.4.1           carData_3.0-5            \n",
       " [43] DBI_1.1.3                 HDF5Array_1.28.1         \n",
       " [45] R.utils_2.12.2            ggsignif_0.6.4           \n",
       " [47] biomaRt_2.56.1            MASS_7.3-60              \n",
       " [49] rappdirs_0.3.3            DelayedArray_0.26.6      \n",
       " [51] rjson_0.2.21              gtools_3.9.4             \n",
       " [53] permute_0.9-7             tools_4.3.2              \n",
       " [55] zip_2.3.0                 R.oo_1.25.0              \n",
       " [57] glue_1.6.2                VennDiagram_1.7.3        \n",
       " [59] restfulr_0.0.15           InteractionSet_1.30.0    \n",
       " [61] rhdf5filters_1.12.1       grid_4.3.2               \n",
       " [63] pbdZMQ_0.3-10             generics_0.1.3           \n",
       " [65] gtable_0.3.4              BSgenome_1.68.0          \n",
       " [67] R.methodsS3_1.8.2         hms_1.1.3                \n",
       " [69] car_3.1-2                 xml2_1.3.5               \n",
       " [71] utf8_1.2.4                XVector_0.40.0           \n",
       " [73] pillar_1.9.0              stringr_1.5.0            \n",
       " [75] IRdisplay_1.1             limma_3.56.2             \n",
       " [77] splines_4.3.2             dplyr_1.1.3              \n",
       " [79] BiocFileCache_2.8.0       lattice_0.22-5           \n",
       " [81] survival_3.5-7            rtracklayer_1.60.0       \n",
       " [83] bit_4.0.5                 tidyselect_1.2.0         \n",
       " [85] RBGL_1.78.0               locfit_1.5-9.8           \n",
       " [87] Biostrings_2.68.1         futile.logger_1.4.3      \n",
       " [89] stringi_1.7.12            yaml_2.3.7               \n",
       " [91] evaluate_0.23             codetools_0.2-19         \n",
       " [93] tibble_3.2.1              graph_1.80.0             \n",
       " [95] cli_3.6.1                 IRkernel_1.3.2           \n",
       " [97] repr_1.1.6                munsell_0.5.0            \n",
       " [99] Rcpp_1.0.11               dbplyr_2.4.0             \n",
       "[101] png_0.1-8                 XML_3.99-0.15            \n",
       "[103] blob_1.2.4                prettyunits_1.2.0        \n",
       "[105] sparseMatrixStats_1.12.2  bitops_1.0-7             \n",
       "[107] scales_1.2.1              purrr_1.0.2              \n",
       "[109] crayon_1.5.2              rlang_1.1.2              \n",
       "[111] KEGGREST_1.40.0           formatR_1.14             "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a85e2abd-3457-47d3-aae0-5f07a7545f13",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "set_plot_dim <- function(\n",
    "    width = 16,\n",
    "    height = 9\n",
    ") {\n",
    "    options(\n",
    "        #jupyter.plot_scale=1,\n",
    "        repr.plot.width = width,\n",
    "        repr.plot.height = height\n",
    "    )\n",
    "}\n",
    "set_plot_dim(5,5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "c6c772dd",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/latex": [
       "'/mnt/Data\\_2/mw-oncodiag'"
      ],
      "text/markdown": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/plain": [
       "[1] \"/mnt/Data_2/mw-oncodiag\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "getwd()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d25c731a-bb17-4f55-8a22-99b61e0d04f7",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    },
    "tags": []
   },
   "source": [
    "### Data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre class=language-r><code>new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = structure(\"samples\", package = \"Biobase\"), \n",
       "<span style=white-space:pre-wrap>    package = \"Biobase\", group = list(), valueClass = character(0), </span>\n",
       "<span style=white-space:pre-wrap>    signature = \"object\", default = NULL, skeleton = (function (object) </span>\n",
       "<span style=white-space:pre-wrap>    stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", </span>\n",
       "<span style=white-space:pre-wrap>        \"samples\"), domain = NA))(object))</span></code></pre>"
      ],
      "text/latex": [
       "\\begin{minted}{r}\n",
       "new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = structure(\"samples\", package = \"Biobase\"), \n",
       "    package = \"Biobase\", group = list(), valueClass = character(0), \n",
       "    signature = \"object\", default = NULL, skeleton = (function (object) \n",
       "    stop(gettextf(\"invalid call in method dispatch to '\\%s' (no default method)\", \n",
       "        \"samples\"), domain = NA))(object))\n",
       "\\end{minted}"
      ],
      "text/markdown": [
       "```r\n",
       "new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = structure(\"samples\", package = \"Biobase\"), \n",
       "    package = \"Biobase\", group = list(), valueClass = character(0), \n",
       "    signature = \"object\", default = NULL, skeleton = (function (object) \n",
       "    stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", \n",
       "        \"samples\"), domain = NA))(object))\n",
       "```"
      ],
      "text/plain": [
       "new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = \"samples\", package = \"Biobase\", \n",
       "    group = list(), valueClass = character(0), signature = \"object\", \n",
       "    default = NULL, skeleton = (function (object) \n",
       "    stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", \n",
       "        \"samples\"), domain = NA))(object))\n",
       "<bytecode: 0x562d469d2db0>\n",
       "<environment: 0x562d469cb948>\n",
       "attr(,\"generic\")\n",
       "[1] \"samples\"\n",
       "attr(,\"generic\")attr(,\"package\")\n",
       "[1] \"Biobase\"\n",
       "attr(,\"package\")\n",
       "[1] \"Biobase\"\n",
       "attr(,\"group\")\n",
       "list()\n",
       "attr(,\"valueClass\")\n",
       "character(0)\n",
       "attr(,\"signature\")\n",
       "[1] \"object\"\n",
       "attr(,\"default\")\n",
       "`\\001NULL\\001`\n",
       "attr(,\"skeleton\")\n",
       "(function (object) \n",
       "stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", \n",
       "    \"samples\"), domain = NA))(object)\n",
       "attr(,\"class\")\n",
       "[1] \"standardGeneric\"\n",
       "attr(,\"class\")attr(,\"package\")\n",
       "[1] \"methods\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "9dc0108b-320d-4f11-9d95-a2c64c472ec5",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 5</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>1</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>2</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>3</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>4</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>5</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>6</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>7</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>8</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>9</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>10</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>11</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>12</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>13</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>14</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>15</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>16</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>21</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>22</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>23</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>24</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>25</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>26</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>27</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>28</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>29</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>30</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 5\n",
       "\\begin{tabular}{r|lllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr>\\\\\n",
       "\\hline\n",
       "\t1 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal \\\\\n",
       "\t2 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral\\\\\n",
       "\t3 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal \\\\\n",
       "\t4 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral\\\\\n",
       "\t5 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal \\\\\n",
       "\t6 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral\\\\\n",
       "\t7 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal \\\\\n",
       "\t8 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral\\\\\n",
       "\t9 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal \\\\\n",
       "\t10 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral\\\\\n",
       "\t11 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal \\\\\n",
       "\t12 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral\\\\\n",
       "\t13 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal \\\\\n",
       "\t14 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral\\\\\n",
       "\t15 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal \\\\\n",
       "\t16 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral\\\\\n",
       "\t21 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal \\\\\n",
       "\t22 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral\\\\\n",
       "\t23 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal \\\\\n",
       "\t24 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral\\\\\n",
       "\t25 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal \\\\\n",
       "\t26 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral\\\\\n",
       "\t27 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal \\\\\n",
       "\t28 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral\\\\\n",
       "\t29 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal \\\\\n",
       "\t30 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 5\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|\n",
       "| 1 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  |\n",
       "| 2 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral |\n",
       "| 3 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  |\n",
       "| 4 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral |\n",
       "| 5 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  |\n",
       "| 6 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral |\n",
       "| 7 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  |\n",
       "| 8 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral |\n",
       "| 9 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  |\n",
       "| 10 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral |\n",
       "| 11 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  |\n",
       "| 12 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral |\n",
       "| 13 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  |\n",
       "| 14 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral |\n",
       "| 15 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  |\n",
       "| 16 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral |\n",
       "| 21 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  |\n",
       "| 22 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral |\n",
       "| 23 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  |\n",
       "| 24 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral |\n",
       "| 25 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  |\n",
       "| 26 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral |\n",
       "| 27 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  |\n",
       "| 28 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral |\n",
       "| 29 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  |\n",
       "| 30 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral |\n",
       "\n"
      ],
      "text/plain": [
       "   sample  fastq_1                                                      \n",
       "1  ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "2  ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "3  ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "4  ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "5  ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "6  ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "7  ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "8  ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "9  ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "10 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "11 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "12 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "13 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "14 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "15 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "16 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "21 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "22 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "23 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "24 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "25 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "26 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "27 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "28 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "29 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "30 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "   fastq_2                                                       patient_number\n",
       "1  inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz   1            \n",
       "2  inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz   1            \n",
       "3  inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz   2            \n",
       "4  inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz   2            \n",
       "5  inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz   3            \n",
       "6  inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz   3            \n",
       "7  inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz   4            \n",
       "8  inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz   4            \n",
       "9  inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz   5            \n",
       "10 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz  5            \n",
       "11 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz  6            \n",
       "12 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz  6            \n",
       "13 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz  7            \n",
       "14 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz  7            \n",
       "15 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz  8            \n",
       "16 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz  8            \n",
       "21 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz 11            \n",
       "22 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz 11            \n",
       "23 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz 12            \n",
       "24 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz 12            \n",
       "25 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz 13            \n",
       "26 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz 13            \n",
       "27 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz 14            \n",
       "28 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz 14            \n",
       "29 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz 15            \n",
       "30 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz 15            \n",
       "   tissue \n",
       "1  normal \n",
       "2  tumoral\n",
       "3  normal \n",
       "4  tumoral\n",
       "5  normal \n",
       "6  tumoral\n",
       "7  normal \n",
       "8  tumoral\n",
       "9  normal \n",
       "10 tumoral\n",
       "11 normal \n",
       "12 tumoral\n",
       "13 normal \n",
       "14 tumoral\n",
       "15 normal \n",
       "16 tumoral\n",
       "21 normal \n",
       "22 tumoral\n",
       "23 normal \n",
       "24 tumoral\n",
       "25 normal \n",
       "26 tumoral\n",
       "27 normal \n",
       "28 tumoral\n",
       "29 normal \n",
       "30 tumoral"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sample_metadata <- read.csv(\n",
    "  \"src/sample_sheets/all.csv\"\n",
    ")\n",
    "# Here I temporarily subset the sample sheets to already available samples\n",
    "suffix <- paste0(\".markdup.sorted_\", context, \".bedGraph\")\n",
    "# input_dir <- \"out/nextflow/nfcore_methylseq/methyldackel/\"\n",
    "input_dir <- \"out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/\"\n",
    "\n",
    "files <- list.files(\n",
    "    path = input_dir,\n",
    "    pattern = suffix\n",
    ")\n",
    "samples <- sub(\n",
    "    suffix,\n",
    "    \"\",\n",
    "    files\n",
    ")\n",
    "sample_metadata <- sample_metadata[sample_metadata$sample %in% samples,]\n",
    "# Remove patient 9 and 10 because they have low coverae\n",
    "sample_metadata <- sample_metadata[!sample_metadata$patient_number %in% c(9,10),]\n",
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "cea6afe8-a07b-486b-9ee9-a1e0d9683807",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# This will be used to load files with read.bismarck()\n",
    "sample_metadata$bedGraph <- paste0(\n",
    "    # 'out/nextflow/nfcore_methylseq/methyldackel/',\n",
    "    # After 2023-11-06, I reprocessed methyldackel outside the nextflow pipeline\n",
    "    # to have the possibility to generate CHH and CHG bedGraph files.\n",
    "    'out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/',\n",
    "    sample_metadata$sample,\n",
    "    '.markdup.sorted_',\n",
    "    context,\n",
    "    '.bedGraph'\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "fa00d848-d751-4ae5-a27a-e57d2c567743",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 6</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th><th scope=col>bedGraph</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ODG_080</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_081</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_082</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_083</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_084</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_085</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_086</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_087</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_088</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_089</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_090</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_091</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_092</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_093</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_094</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_095</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_100</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_101</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_102</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_103</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_104</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_105</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_106</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_107</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_108</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_109</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 6\n",
       "\\begin{tabular}{r|llllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue & bedGraph\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\tODG\\_080 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_080.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_081 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_081.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_082 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_082.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_083 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_083.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_084 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_084.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_085 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_085.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_086 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_086.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_087 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_087.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_088 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_088.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_089 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_089.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_090 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_090.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_091 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_091.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_092 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_092.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_093 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_093.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_094 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_094.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_095 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_095.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_100 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_100.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_101 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_101.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_102 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_102.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_103 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_103.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_104 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_104.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_105 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_105.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_106 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_106.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_107 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_107.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_108 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_108.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_109 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_109.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 6\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; | bedGraph &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|\n",
       "| ODG_080 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_081 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_082 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_083 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_084 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_085 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_086 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_087 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_088 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_089 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_090 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_091 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_092 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_093 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_094 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_095 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_100 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_101 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_102 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_103 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_104 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_105 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_106 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_107 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_108 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_109 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph |\n",
       "\n"
      ],
      "text/plain": [
       "        sample  fastq_1                                                      \n",
       "ODG_080 ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "ODG_081 ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "ODG_082 ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "ODG_083 ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "ODG_084 ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "ODG_085 ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "ODG_086 ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "ODG_087 ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "ODG_088 ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "ODG_089 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "ODG_090 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "ODG_091 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "ODG_092 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "ODG_093 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "ODG_094 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "ODG_095 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "ODG_100 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "ODG_101 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "ODG_102 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "ODG_103 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "ODG_104 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "ODG_105 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "ODG_106 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "ODG_107 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "ODG_108 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "ODG_109 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "        fastq_2                                                      \n",
       "ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz \n",
       "ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz \n",
       "ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz \n",
       "ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz \n",
       "ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz \n",
       "ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz \n",
       "ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz \n",
       "ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz \n",
       "ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz \n",
       "ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz\n",
       "ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz\n",
       "ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz\n",
       "ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz\n",
       "ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz\n",
       "ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz\n",
       "ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz\n",
       "ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz\n",
       "ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz\n",
       "ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz\n",
       "ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz\n",
       "ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz\n",
       "ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz\n",
       "ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz\n",
       "ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz\n",
       "ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz\n",
       "ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz\n",
       "        patient_number tissue \n",
       "ODG_080  1             normal \n",
       "ODG_081  1             tumoral\n",
       "ODG_082  2             normal \n",
       "ODG_083  2             tumoral\n",
       "ODG_084  3             normal \n",
       "ODG_085  3             tumoral\n",
       "ODG_086  4             normal \n",
       "ODG_087  4             tumoral\n",
       "ODG_088  5             normal \n",
       "ODG_089  5             tumoral\n",
       "ODG_090  6             normal \n",
       "ODG_091  6             tumoral\n",
       "ODG_092  7             normal \n",
       "ODG_093  7             tumoral\n",
       "ODG_094  8             normal \n",
       "ODG_095  8             tumoral\n",
       "ODG_100 11             normal \n",
       "ODG_101 11             tumoral\n",
       "ODG_102 12             normal \n",
       "ODG_103 12             tumoral\n",
       "ODG_104 13             normal \n",
       "ODG_105 13             tumoral\n",
       "ODG_106 14             normal \n",
       "ODG_107 14             tumoral\n",
       "ODG_108 15             normal \n",
       "ODG_109 15             tumoral\n",
       "        bedGraph                                                                                                                                 \n",
       "ODG_080 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph\n",
       "ODG_081 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph\n",
       "ODG_082 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph\n",
       "ODG_083 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph\n",
       "ODG_084 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph\n",
       "ODG_085 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph\n",
       "ODG_086 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph\n",
       "ODG_087 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph\n",
       "ODG_088 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph\n",
       "ODG_089 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph\n",
       "ODG_090 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph\n",
       "ODG_091 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph\n",
       "ODG_092 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph\n",
       "ODG_093 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph\n",
       "ODG_094 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph\n",
       "ODG_095 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph\n",
       "ODG_100 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph\n",
       "ODG_101 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph\n",
       "ODG_102 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph\n",
       "ODG_103 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph\n",
       "ODG_104 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph\n",
       "ODG_105 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph\n",
       "ODG_106 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph\n",
       "ODG_107 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph\n",
       "ODG_108 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph\n",
       "ODG_109 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# row.names should be declared to be used as row.names of the BSseq object\n",
    "# because DMLtest() uses these row.namesto select samples\n",
    "row.names(sample_metadata) <- sample_metadata$sample\n",
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'out/ipynb/compute_unsupervised_classification/pca.rds'"
      ],
      "text/latex": [
       "'out/ipynb/compute\\_unsupervised\\_classification/pca.rds'"
      ],
      "text/markdown": [
       "'out/ipynb/compute_unsupervised_classification/pca.rds'"
      ],
      "text/plain": [
       "[1] \"out/ipynb/compute_unsupervised_classification/pca.rds\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    " snakemake@log$rds"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/latex": [
       "'/mnt/Data\\_2/mw-oncodiag'"
      ],
      "text/markdown": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/plain": [
       "[1] \"/mnt/Data_2/mw-oncodiag\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "getwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'CHH'"
      ],
      "text/latex": [
       "'CHH'"
      ],
      "text/markdown": [
       "'CHH'"
      ],
      "text/plain": [
       "[1] \"CHH\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "context"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# loci <- findLoci(\n",
    "#     pattern = sub(\"p\",\"\", context),\n",
    "#     subject = \"out/gunzip/to-stdout/wget/ftp/ftp.ensembl.org/pub/grch37/current/fasta/homo_sapiens/dna/Homo_sapiens.GRCh37.dna.toplevel.fasta\",\n",
    "#     # subject = BSgenome.Hsapiens.UCSC.hg19,\n",
    "#     # include = seqlevels(subject),\n",
    "#     # strand = c(\"*\", \"+\", \"-\"),\n",
    "#     strand = \"*\",\n",
    "#     # fixed = \"subject\",\n",
    "#     resize = TRUE\n",
    "# )\n",
    "# seqlevels(loci) <- sub(\" dna.*$\", \"\", seqlevels(loci))\n",
    "# loci[seqnames(loci) == 1,]\n",
    "\n",
    "# loci"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# loci_minus <- findLoci(\n",
    "#     pattern = sub(\"p\",\"\", context),\n",
    "#     subject = \"out/gunzip/to-stdout/wget/ftp/ftp.ensembl.org/pub/grch37/current/fasta/homo_sapiens/dna/Homo_sapiens.GRCh37.dna.toplevel.fasta\",\n",
    "#     # subject = BSgenome.Hsapiens.UCSC.hg19,\n",
    "#     # include = seqlevels(subject),\n",
    "#     # strand = c(\"*\", \"+\", \"-\"),\n",
    "#     strand = \"-\",\n",
    "#     # fixed = \"subject\",\n",
    "#     resize = TRUE\n",
    "# )\n",
    "# seqlevels(loci_minus) <- sub(\" dna.*$\", \"\", seqlevels(loci_minus))\n",
    "# loci_minus[seqnames(loci_minus) == 1,]\n",
    "\n",
    "# loci"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# loci_plus <- findLoci(\n",
    "#     pattern = sub(\"p\",\"\", context),\n",
    "#     subject = \"out/gunzip/to-stdout/wget/ftp/ftp.ensembl.org/pub/grch37/current/fasta/homo_sapiens/dna/Homo_sapiens.GRCh37.dna.toplevel.fasta\",\n",
    "#     # subject = BSgenome.Hsapiens.UCSC.hg19,\n",
    "#     # include = seqlevels(subject),\n",
    "#     # strand = c(\"*\", \"+\", \"-\"),\n",
    "#     strand = \"+\",\n",
    "#     # fixed = \"subject\",\n",
    "#     resize = TRUE\n",
    "# )\n",
    "# seqlevels(loci_plus) <- sub(\" dna.*$\", \"\", seqlevels(loci_plus))\n",
    "# loci_plus[seqnames(loci_plus) == 1,]\n",
    "\n",
    "# # loci"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 6</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th><th scope=col>bedGraph</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ODG_080</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_081</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_082</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_083</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_084</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_085</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_086</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_087</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_088</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_089</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_090</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_091</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_092</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_093</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_094</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_095</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_100</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_101</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_102</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_103</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_104</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_105</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_106</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_107</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_108</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_109</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 6\n",
       "\\begin{tabular}{r|llllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue & bedGraph\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\tODG\\_080 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_080.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_081 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_081.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_082 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_082.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_083 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_083.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_084 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_084.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_085 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_085.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_086 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_086.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_087 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_087.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_088 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_088.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_089 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_089.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_090 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_090.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_091 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_091.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_092 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_092.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_093 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_093.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_094 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_094.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_095 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_095.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_100 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_100.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_101 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_101.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_102 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_102.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_103 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_103.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_104 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_104.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_105 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_105.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_106 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_106.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_107 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_107.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_108 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_108.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\tODG\\_109 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_109.markdup.sorted\\_CHH.bedGraph\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 6\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; | bedGraph &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|\n",
       "| ODG_080 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_081 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_082 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_083 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_084 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_085 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_086 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_087 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_088 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_089 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_090 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_091 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_092 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_093 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_094 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_095 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_100 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_101 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_102 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_103 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_104 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_105 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_106 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_107 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_108 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph |\n",
       "| ODG_109 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph |\n",
       "\n"
      ],
      "text/plain": [
       "        sample  fastq_1                                                      \n",
       "ODG_080 ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "ODG_081 ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "ODG_082 ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "ODG_083 ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "ODG_084 ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "ODG_085 ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "ODG_086 ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "ODG_087 ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "ODG_088 ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "ODG_089 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "ODG_090 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "ODG_091 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "ODG_092 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "ODG_093 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "ODG_094 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "ODG_095 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "ODG_100 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "ODG_101 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "ODG_102 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "ODG_103 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "ODG_104 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "ODG_105 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "ODG_106 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "ODG_107 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "ODG_108 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "ODG_109 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "        fastq_2                                                      \n",
       "ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz \n",
       "ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz \n",
       "ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz \n",
       "ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz \n",
       "ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz \n",
       "ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz \n",
       "ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz \n",
       "ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz \n",
       "ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz \n",
       "ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz\n",
       "ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz\n",
       "ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz\n",
       "ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz\n",
       "ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz\n",
       "ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz\n",
       "ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz\n",
       "ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz\n",
       "ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz\n",
       "ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz\n",
       "ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz\n",
       "ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz\n",
       "ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz\n",
       "ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz\n",
       "ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz\n",
       "ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz\n",
       "ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz\n",
       "        patient_number tissue \n",
       "ODG_080  1             normal \n",
       "ODG_081  1             tumoral\n",
       "ODG_082  2             normal \n",
       "ODG_083  2             tumoral\n",
       "ODG_084  3             normal \n",
       "ODG_085  3             tumoral\n",
       "ODG_086  4             normal \n",
       "ODG_087  4             tumoral\n",
       "ODG_088  5             normal \n",
       "ODG_089  5             tumoral\n",
       "ODG_090  6             normal \n",
       "ODG_091  6             tumoral\n",
       "ODG_092  7             normal \n",
       "ODG_093  7             tumoral\n",
       "ODG_094  8             normal \n",
       "ODG_095  8             tumoral\n",
       "ODG_100 11             normal \n",
       "ODG_101 11             tumoral\n",
       "ODG_102 12             normal \n",
       "ODG_103 12             tumoral\n",
       "ODG_104 13             normal \n",
       "ODG_105 13             tumoral\n",
       "ODG_106 14             normal \n",
       "ODG_107 14             tumoral\n",
       "ODG_108 15             normal \n",
       "ODG_109 15             tumoral\n",
       "        bedGraph                                                                                                                                 \n",
       "ODG_080 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph\n",
       "ODG_081 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph\n",
       "ODG_082 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph\n",
       "ODG_083 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph\n",
       "ODG_084 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph\n",
       "ODG_085 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph\n",
       "ODG_086 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph\n",
       "ODG_087 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph\n",
       "ODG_088 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph\n",
       "ODG_089 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph\n",
       "ODG_090 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph\n",
       "ODG_091 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph\n",
       "ODG_092 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph\n",
       "ODG_093 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph\n",
       "ODG_094 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph\n",
       "ODG_095 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph\n",
       "ODG_100 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph\n",
       "ODG_101 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph\n",
       "ODG_102 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph\n",
       "ODG_103 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph\n",
       "ODG_104 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph\n",
       "ODG_105 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph\n",
       "ODG_106 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph\n",
       "ODG_107 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph\n",
       "ODG_108 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph\n",
       "ODG_109 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "outdir <- \"out/ipynb/compute_simple_stats\"\n",
    "outfile <- file.path(\n",
    "    outdir,\n",
    "    paste0(\n",
    "        context,\n",
    "        \".rds\"\n",
    "    )\n",
    ")\n",
    "\n",
    "if (file.exists(outfile)) {\n",
    "    sample_metadata <-readRDS(outfile)\n",
    "} else {\n",
    "    # We create an output sample_metadata that will hold the results\n",
    "    sample_metadata$mean_methylation <- NA\n",
    "\n",
    "    dir.create(outdir)\n",
    "    saveRDS(\n",
    "        sample_metadata,\n",
    "        file = outfile\n",
    "    )\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style>\n",
       ".list-inline {list-style: none; margin:0; padding: 0}\n",
       ".list-inline>li {display: inline-block}\n",
       ".list-inline>li:not(:last-child)::after {content: \"\\00b7\"; padding: 0 .5ex}\n",
       "</style>\n",
       "<ol class=list-inline><li>0.685222739898728</li><li>0.765369276238975</li><li>0.655039166095789</li><li>0.774670610727703</li><li>0.839609371642928</li><li>0.850248203902655</li><li>0.79021925736339</li><li>2.94059124135689</li><li>0.696793856743372</li><li>0.7560836764059</li><li>0.756859967244221</li><li>0.756072823454773</li><li>0.780221669185831</li><li>0.790557805354149</li><li>0.704822337810595</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li></ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 0.685222739898728\n",
       "\\item 0.765369276238975\n",
       "\\item 0.655039166095789\n",
       "\\item 0.774670610727703\n",
       "\\item 0.839609371642928\n",
       "\\item 0.850248203902655\n",
       "\\item 0.79021925736339\n",
       "\\item 2.94059124135689\n",
       "\\item 0.696793856743372\n",
       "\\item 0.7560836764059\n",
       "\\item 0.756859967244221\n",
       "\\item 0.756072823454773\n",
       "\\item 0.780221669185831\n",
       "\\item 0.790557805354149\n",
       "\\item 0.704822337810595\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 0.685222739898728\n",
       "2. 0.765369276238975\n",
       "3. 0.655039166095789\n",
       "4. 0.774670610727703\n",
       "5. 0.839609371642928\n",
       "6. 0.850248203902655\n",
       "7. 0.79021925736339\n",
       "8. 2.94059124135689\n",
       "9. 0.696793856743372\n",
       "10. 0.7560836764059\n",
       "11. 0.756859967244221\n",
       "12. 0.756072823454773\n",
       "13. 0.780221669185831\n",
       "14. 0.790557805354149\n",
       "15. 0.704822337810595\n",
       "16. &lt;NA&gt;\n",
       "17. &lt;NA&gt;\n",
       "18. &lt;NA&gt;\n",
       "19. &lt;NA&gt;\n",
       "20. &lt;NA&gt;\n",
       "21. &lt;NA&gt;\n",
       "22. &lt;NA&gt;\n",
       "23. &lt;NA&gt;\n",
       "24. &lt;NA&gt;\n",
       "25. &lt;NA&gt;\n",
       "26. &lt;NA&gt;\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       " [1] 0.6852227 0.7653693 0.6550392 0.7746706 0.8396094 0.8502482 0.7902193\n",
       " [8] 2.9405912 0.6967939 0.7560837 0.7568600 0.7560728 0.7802217 0.7905578\n",
       "[15] 0.7048223        NA        NA        NA        NA        NA        NA\n",
       "[22]        NA        NA        NA        NA        NA"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sample_metadata$mean_methylation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading sample 1/26\n",
      "\n",
      "Loading sample 2/26\n",
      "\n",
      "Loading sample 3/26\n",
      "\n",
      "Loading sample 4/26\n",
      "\n",
      "Loading sample 5/26\n",
      "\n",
      "Loading sample 6/26\n",
      "\n",
      "Loading sample 7/26\n",
      "\n",
      "Loading sample 8/26\n",
      "\n",
      "Loading sample 9/26\n",
      "\n",
      "Loading sample 10/26\n",
      "\n",
      "Loading sample 11/26\n",
      "\n",
      "Loading sample 12/26\n",
      "\n",
      "Loading sample 13/26\n",
      "\n",
      "Loading sample 14/26\n",
      "\n",
      "Loading sample 15/26\n",
      "\n",
      "Loading sample 16/26\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading sample 17/26\n",
      "\n",
      "Loading sample 18/26\n",
      "\n",
      "Loading sample 19/26\n",
      "\n",
      "Loading sample 20/26\n",
      "\n",
      "Loading sample 21/26\n",
      "\n",
      "Loading sample 22/26\n",
      "\n",
      "Loading sample 23/26\n",
      "\n",
      "Loading sample 24/26\n",
      "\n",
      "Loading sample 25/26\n",
      "\n",
      "Loading sample 26/26\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 7</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th><th scope=col>bedGraph</th><th scope=col>mean_methylation</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ODG_080</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph</td><td>0.6852227</td></tr>\n",
       "\t<tr><th scope=row>ODG_081</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph</td><td>0.7653693</td></tr>\n",
       "\t<tr><th scope=row>ODG_082</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph</td><td>0.6550392</td></tr>\n",
       "\t<tr><th scope=row>ODG_083</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph</td><td>0.7746706</td></tr>\n",
       "\t<tr><th scope=row>ODG_084</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph</td><td>0.8396094</td></tr>\n",
       "\t<tr><th scope=row>ODG_085</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph</td><td>0.8502482</td></tr>\n",
       "\t<tr><th scope=row>ODG_086</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph</td><td>0.7902193</td></tr>\n",
       "\t<tr><th scope=row>ODG_087</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph</td><td>2.9405912</td></tr>\n",
       "\t<tr><th scope=row>ODG_088</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph</td><td>0.6967939</td></tr>\n",
       "\t<tr><th scope=row>ODG_089</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph</td><td>0.7560837</td></tr>\n",
       "\t<tr><th scope=row>ODG_090</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph</td><td>0.7568600</td></tr>\n",
       "\t<tr><th scope=row>ODG_091</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph</td><td>0.7560728</td></tr>\n",
       "\t<tr><th scope=row>ODG_092</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph</td><td>0.7802217</td></tr>\n",
       "\t<tr><th scope=row>ODG_093</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph</td><td>0.7905578</td></tr>\n",
       "\t<tr><th scope=row>ODG_094</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph</td><td>0.7048223</td></tr>\n",
       "\t<tr><th scope=row>ODG_095</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph</td><td>0.6710358</td></tr>\n",
       "\t<tr><th scope=row>ODG_100</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph</td><td>0.6552023</td></tr>\n",
       "\t<tr><th scope=row>ODG_101</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph</td><td>0.9500158</td></tr>\n",
       "\t<tr><th scope=row>ODG_102</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph</td><td>0.6446235</td></tr>\n",
       "\t<tr><th scope=row>ODG_103</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph</td><td>1.8708932</td></tr>\n",
       "\t<tr><th scope=row>ODG_104</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph</td><td>1.5533082</td></tr>\n",
       "\t<tr><th scope=row>ODG_105</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph</td><td>1.8955459</td></tr>\n",
       "\t<tr><th scope=row>ODG_106</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph</td><td>0.6437558</td></tr>\n",
       "\t<tr><th scope=row>ODG_107</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph</td><td>1.7055948</td></tr>\n",
       "\t<tr><th scope=row>ODG_108</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph</td><td>1.9538436</td></tr>\n",
       "\t<tr><th scope=row>ODG_109</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph</td><td>3.4975316</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 7\n",
       "\\begin{tabular}{r|lllllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue & bedGraph & mean\\_methylation\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr> & <chr> & <dbl>\\\\\n",
       "\\hline\n",
       "\tODG\\_080 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_080.markdup.sorted\\_CHH.bedGraph & 0.6852227\\\\\n",
       "\tODG\\_081 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_081.markdup.sorted\\_CHH.bedGraph & 0.7653693\\\\\n",
       "\tODG\\_082 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_082.markdup.sorted\\_CHH.bedGraph & 0.6550392\\\\\n",
       "\tODG\\_083 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_083.markdup.sorted\\_CHH.bedGraph & 0.7746706\\\\\n",
       "\tODG\\_084 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_084.markdup.sorted\\_CHH.bedGraph & 0.8396094\\\\\n",
       "\tODG\\_085 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_085.markdup.sorted\\_CHH.bedGraph & 0.8502482\\\\\n",
       "\tODG\\_086 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_086.markdup.sorted\\_CHH.bedGraph & 0.7902193\\\\\n",
       "\tODG\\_087 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_087.markdup.sorted\\_CHH.bedGraph & 2.9405912\\\\\n",
       "\tODG\\_088 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_088.markdup.sorted\\_CHH.bedGraph & 0.6967939\\\\\n",
       "\tODG\\_089 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_089.markdup.sorted\\_CHH.bedGraph & 0.7560837\\\\\n",
       "\tODG\\_090 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_090.markdup.sorted\\_CHH.bedGraph & 0.7568600\\\\\n",
       "\tODG\\_091 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_091.markdup.sorted\\_CHH.bedGraph & 0.7560728\\\\\n",
       "\tODG\\_092 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_092.markdup.sorted\\_CHH.bedGraph & 0.7802217\\\\\n",
       "\tODG\\_093 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_093.markdup.sorted\\_CHH.bedGraph & 0.7905578\\\\\n",
       "\tODG\\_094 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_094.markdup.sorted\\_CHH.bedGraph & 0.7048223\\\\\n",
       "\tODG\\_095 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_095.markdup.sorted\\_CHH.bedGraph & 0.6710358\\\\\n",
       "\tODG\\_100 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_100.markdup.sorted\\_CHH.bedGraph & 0.6552023\\\\\n",
       "\tODG\\_101 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_101.markdup.sorted\\_CHH.bedGraph & 0.9500158\\\\\n",
       "\tODG\\_102 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_102.markdup.sorted\\_CHH.bedGraph & 0.6446235\\\\\n",
       "\tODG\\_103 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_103.markdup.sorted\\_CHH.bedGraph & 1.8708932\\\\\n",
       "\tODG\\_104 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_104.markdup.sorted\\_CHH.bedGraph & 1.5533082\\\\\n",
       "\tODG\\_105 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_105.markdup.sorted\\_CHH.bedGraph & 1.8955459\\\\\n",
       "\tODG\\_106 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_106.markdup.sorted\\_CHH.bedGraph & 0.6437558\\\\\n",
       "\tODG\\_107 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_107.markdup.sorted\\_CHH.bedGraph & 1.7055948\\\\\n",
       "\tODG\\_108 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_108.markdup.sorted\\_CHH.bedGraph & 1.9538436\\\\\n",
       "\tODG\\_109 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_109.markdup.sorted\\_CHH.bedGraph & 3.4975316\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 7\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; | bedGraph &lt;chr&gt; | mean_methylation &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|\n",
       "| ODG_080 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph | 0.6852227 |\n",
       "| ODG_081 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph | 0.7653693 |\n",
       "| ODG_082 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph | 0.6550392 |\n",
       "| ODG_083 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph | 0.7746706 |\n",
       "| ODG_084 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph | 0.8396094 |\n",
       "| ODG_085 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph | 0.8502482 |\n",
       "| ODG_086 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph | 0.7902193 |\n",
       "| ODG_087 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph | 2.9405912 |\n",
       "| ODG_088 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph | 0.6967939 |\n",
       "| ODG_089 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph | 0.7560837 |\n",
       "| ODG_090 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph | 0.7568600 |\n",
       "| ODG_091 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph | 0.7560728 |\n",
       "| ODG_092 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph | 0.7802217 |\n",
       "| ODG_093 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph | 0.7905578 |\n",
       "| ODG_094 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph | 0.7048223 |\n",
       "| ODG_095 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph | 0.6710358 |\n",
       "| ODG_100 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph | 0.6552023 |\n",
       "| ODG_101 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph | 0.9500158 |\n",
       "| ODG_102 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph | 0.6446235 |\n",
       "| ODG_103 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph | 1.8708932 |\n",
       "| ODG_104 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph | 1.5533082 |\n",
       "| ODG_105 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph | 1.8955459 |\n",
       "| ODG_106 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph | 0.6437558 |\n",
       "| ODG_107 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph | 1.7055948 |\n",
       "| ODG_108 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph | 1.9538436 |\n",
       "| ODG_109 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph | 3.4975316 |\n",
       "\n"
      ],
      "text/plain": [
       "        sample  fastq_1                                                      \n",
       "ODG_080 ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "ODG_081 ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "ODG_082 ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "ODG_083 ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "ODG_084 ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "ODG_085 ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "ODG_086 ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "ODG_087 ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "ODG_088 ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "ODG_089 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "ODG_090 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "ODG_091 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "ODG_092 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "ODG_093 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "ODG_094 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "ODG_095 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "ODG_100 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "ODG_101 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "ODG_102 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "ODG_103 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "ODG_104 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "ODG_105 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "ODG_106 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "ODG_107 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "ODG_108 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "ODG_109 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "        fastq_2                                                      \n",
       "ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz \n",
       "ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz \n",
       "ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz \n",
       "ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz \n",
       "ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz \n",
       "ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz \n",
       "ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz \n",
       "ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz \n",
       "ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz \n",
       "ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz\n",
       "ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz\n",
       "ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz\n",
       "ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz\n",
       "ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz\n",
       "ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz\n",
       "ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz\n",
       "ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz\n",
       "ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz\n",
       "ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz\n",
       "ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz\n",
       "ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz\n",
       "ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz\n",
       "ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz\n",
       "ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz\n",
       "ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz\n",
       "ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz\n",
       "        patient_number tissue \n",
       "ODG_080  1             normal \n",
       "ODG_081  1             tumoral\n",
       "ODG_082  2             normal \n",
       "ODG_083  2             tumoral\n",
       "ODG_084  3             normal \n",
       "ODG_085  3             tumoral\n",
       "ODG_086  4             normal \n",
       "ODG_087  4             tumoral\n",
       "ODG_088  5             normal \n",
       "ODG_089  5             tumoral\n",
       "ODG_090  6             normal \n",
       "ODG_091  6             tumoral\n",
       "ODG_092  7             normal \n",
       "ODG_093  7             tumoral\n",
       "ODG_094  8             normal \n",
       "ODG_095  8             tumoral\n",
       "ODG_100 11             normal \n",
       "ODG_101 11             tumoral\n",
       "ODG_102 12             normal \n",
       "ODG_103 12             tumoral\n",
       "ODG_104 13             normal \n",
       "ODG_105 13             tumoral\n",
       "ODG_106 14             normal \n",
       "ODG_107 14             tumoral\n",
       "ODG_108 15             normal \n",
       "ODG_109 15             tumoral\n",
       "        bedGraph                                                                                                                                 \n",
       "ODG_080 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CHH.bedGraph\n",
       "ODG_081 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CHH.bedGraph\n",
       "ODG_082 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CHH.bedGraph\n",
       "ODG_083 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CHH.bedGraph\n",
       "ODG_084 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CHH.bedGraph\n",
       "ODG_085 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CHH.bedGraph\n",
       "ODG_086 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CHH.bedGraph\n",
       "ODG_087 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CHH.bedGraph\n",
       "ODG_088 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CHH.bedGraph\n",
       "ODG_089 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CHH.bedGraph\n",
       "ODG_090 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CHH.bedGraph\n",
       "ODG_091 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CHH.bedGraph\n",
       "ODG_092 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CHH.bedGraph\n",
       "ODG_093 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CHH.bedGraph\n",
       "ODG_094 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CHH.bedGraph\n",
       "ODG_095 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CHH.bedGraph\n",
       "ODG_100 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CHH.bedGraph\n",
       "ODG_101 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CHH.bedGraph\n",
       "ODG_102 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CHH.bedGraph\n",
       "ODG_103 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CHH.bedGraph\n",
       "ODG_104 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CHH.bedGraph\n",
       "ODG_105 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CHH.bedGraph\n",
       "ODG_106 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CHH.bedGraph\n",
       "ODG_107 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CHH.bedGraph\n",
       "ODG_108 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CHH.bedGraph\n",
       "ODG_109 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CHH.bedGraph\n",
       "        mean_methylation\n",
       "ODG_080 0.6852227       \n",
       "ODG_081 0.7653693       \n",
       "ODG_082 0.6550392       \n",
       "ODG_083 0.7746706       \n",
       "ODG_084 0.8396094       \n",
       "ODG_085 0.8502482       \n",
       "ODG_086 0.7902193       \n",
       "ODG_087 2.9405912       \n",
       "ODG_088 0.6967939       \n",
       "ODG_089 0.7560837       \n",
       "ODG_090 0.7568600       \n",
       "ODG_091 0.7560728       \n",
       "ODG_092 0.7802217       \n",
       "ODG_093 0.7905578       \n",
       "ODG_094 0.7048223       \n",
       "ODG_095 0.6710358       \n",
       "ODG_100 0.6552023       \n",
       "ODG_101 0.9500158       \n",
       "ODG_102 0.6446235       \n",
       "ODG_103 1.8708932       \n",
       "ODG_104 1.5533082       \n",
       "ODG_105 1.8955459       \n",
       "ODG_106 0.6437558       \n",
       "ODG_107 1.7055948       \n",
       "ODG_108 1.9538436       \n",
       "ODG_109 3.4975316       "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for (row in 1:nrow(sample_metadata)) {\n",
    "    message(\"Loading sample \", row, \"/\", nrow(sample_metadata))\n",
    "    if (is.na(sample_metadata[row,]$mean_methylation)) {\n",
    "        d <- fread(\n",
    "            sample_metadata[row,]$bedGraph,\n",
    "            select = 4,\n",
    "            skip = 1,\n",
    "            sep = \"\\t\"\n",
    "        )\n",
    "        sample_metadata[row,]$mean_methylation <- mean(d$V4)\n",
    "        rm(d)\n",
    "        saveRDS(\n",
    "            sample_metadata,\n",
    "            file = outfile\n",
    "        )\n",
    "    }\n",
    "\n",
    "}\n",
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlgAAAJYCAMAAACJuGjuAAADAFBMVEUAAAAAAP8BAQECAgIDAwMEBAQFBQUGBgYHBwcICAgJCQkKCgoLCwsMDAwNDQ0ODg4PDw8QEBARERESEhITExMUFBQVFRUWFhYXFxcYGBgZGRkaGhobGxscHBwdHR0eHh4fHx8gICAhISEiIiIjIyMkJCQlJSUmJiYnJycoKCgpKSkqKiorKyssLCwtLS0uLi4vLy8wMDAxMTEyMjIzMzM0NDQ1NTU2NjY3Nzc4ODg5OTk6Ojo7Ozs8PDw9PT0+Pj4/Pz9AQEBBQUFCQkJDQ0NERERFRUVGRkZHR0dISEhJSUlKSkpLS0tMTExNTU1OTk5PT09QUFBRUVFSUlJTU1NUVFRVVVVWVlZXV1dYWFhZWVlaWlpbW1tcXFxdXV1eXl5fX19gYGBhYWFiYmJjY2NkZGRlZWVmZmZnZ2doaGhpaWlqampra2tsbGxubm5vb29wcHBxcXFycnJzc3N0dHR1dXV2dnZ3d3d4eHh5eXl6enp7e3t8fHx9fX1+fn5/f3+AgICBgYGCgoKDg4OEhISFhYWGhoaHh4eIiIiJiYmKioqLi4uMjIyNjY2Ojo6Pj4+QkJCRkZGSkpKTk5OUlJSVlZWWlpaXl5eYmJiZmZmampqbm5udnZ2enp6fn5+goKChoaGioqKjo6OkpKSlpaWmpqanp6eoqKipqamqqqqrq6usrKytra2urq6vr6+wsLCxsbGysrKzs7O0tLS1tbW2tra3t7e4uLi5ubm6urq7u7u8vLy9vb2+vr6/v7/AwMDBwcHCwsLDw8PExMTFxcXGxsbHx8fIyMjJycnKysrLy8vMzMzNzc3Ozs7Pz8/Q0NDR0dHS0tLT09PU1NTV1dXW1tbX19fY2NjZ2dna2trb29vc3Nzd3d3e3t7f39/g4ODh4eHi4uLj4+Pk5OTl5eXm5ubn5+fo6Ojp6enq6urr6+vs7Ozt7e3u7u7v7+/w8PDx8fHy8vLz8/P09PT19fX29vb39/f4+Pj5+fn6+vr7+/v8/Pz9/f3+/v7/AAD///9u/oFqAAAACXBIWXMAABJ0AAASdAHeZh94AAAgAElEQVR4nO3dC3wU1dnH8WOIINiKVRFtEa+tttLWova1LVW0SlU0VG7aiteitSqlFlus4hstCiIqVFsBrdQXvFVahAJeqCLeERBEaABBAbUBZnezScj90vPObMjZOc9mz2SdZ2Em8/99PiaTPZmTzTlf3CVkd4VEKA+JvX0FUMcMsFBeAiyUlwAL5SXAQnkJsFBeAiyUlwAL5aW8wKpN1BlGmxMVppMrE02G0bpEjenkRLlptCrRaBhtSFSZTk6WmUarE/WG0abELtPJFQnTaG2i1jDqsZp7rbzAqrGMS2ElTSeXWyZYtVa16eSYcfcrrQbDaL1l3P1E3DRaZZn+LDVaxt1PWqbRGsv0Z8ljNfdagKUCLM4ASwVYnAGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDgDLBVgcQZYKsDiDLBUgMUZYKkAizPAUgEWZ4ClAizOAEsFWJwBlgqwOAMsFWBxBlgqwOKMA1ZDPWmXVUUvclVnlRlG68usWsNolVVpOjmWMI2WWzWG0WqrwnRyPG4arTR+y7VW0nRymWUa3WXtMoy2tZoMe+o7DlhVFaSklaQXubNiptG4VW4YLbPKjFPHTaMJ4/VKWgnTyTHjtS4zTl1uvl5xyzx1rt9yM8Om+g03haqw3hT+V8s0054MsFSAxRlgqQCLM8BSARZngKUCLM4ASwVYnAGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDgDLBVgcQZYKsDiDLBUgMUZYKkAizPAUgEWZ4ClAizOAEsFWJwBlgqwOAMsFWBxBlgqwOIMsFSAxRlgqQCLM8BSARZngKUCLM4ASwVYnAGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDgDLBVgcQZYKsDiDLBUgMUZYKkAizPAUgEWZ4ClAizOAEsFWJwBlgqwOAMsFWBxBlgqwOIMsFSAxRlgqQCLM8BSARZngKUCLM4ASwVYnAGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDjzhhVb+FwpPf70n4tMOwhYJMDKbP3wceOGrtSP3xoy/tafbsx+DmCRACuz0TOknHlNs/u4ccQcKaf8Lvs5gEUCrIxKi7bY21W00X2cuDch5UvDs58EWCTAymjZoEb77fCX6bGc/NvsJwEWCbAyeiX1P6ar5pPjl269aqv6nIoEKW7F6UXuLMs0GjOOek0dM09tOjnucbJx1Hy9vKb2+JaNJxNYzkXNecKSS16wXk5hunIBOf7HjMumqqtfHqNZGZdoo8ZhP6M+v7Cvr7zXrrYOy7koDLBabv4ufoUey08GvZn1JNwUknBTmFFp0SdSxos2u493zHf+SFz6bNaTAIsEWJmNmi7l49c2yeYmdfzZoDekXFm0Nus5gEUCrMzWDv397UNWSznxpvTxM0MmjB88K/s5gEUCrDbaMW/+dvvd6y+kj+VH8xduNZwCWCTA4gmwSIDFE2CRAIunvMIyLiNgmWbakwGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDgDLBVgcQZYKsDiDLBUgMUZYKkAizPAUgEWZ4ClAizOAEsFWJwBlgqwOAMsFWBxBlgqwOIMsFSAxRlgqQCLM8BSARZngKUCLM4ASwVYnAGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDgDLBVgcQZYKsDiDLBUgMUZYKkAizPAUgEWZ4ClAizOAEsFWJwBlgqwOAMsFWBxBlgqwOIMsFSAxRlgqQCLM8BSARZngKUCLM4ASwVYnAGWCrA4AywVYHEGWCrA4gywVIDFGWCpAIszwFIBFmeApQIszgBLBVicAZYKsDgDLBVgcQZYKsDiDLBUgMUZB6zqSlLSStKL3Flx02jcqjCMJvV1pMMx49QJq9wwWm6VmU6OxUyjZcZvucJKmE6OW6ZR82pWEFjOZc0Mm+o3Dlj1taRKaxe9yFWNlTCM1pZZNYbRSn0d6XDMOHXSqjKMVlnlppPjMdNohfFbrraSppMTlmm00qo0jNYQWM5lDHvqO9wUqnBTyBlgqQCLM8BSARZngKUCLM4ASwVYnAGWCrA4AywVYHGWhpW0Gu3/Wqv3MylgkSINq78osf9rbYmfSQGLFGlYM8butP9rbbOfSQGLFGlYu2u0v88qn5MCFgmwlp70mJSP7T+20dekgEWKPKz1XY5YJOV7l4gxviYFLFLkYf3qsFjqfXFn40J4BVikyMM6/6ct79eJFX4mBSxS5GH97NSW9y+IdX4mBSxS5GHNFpOcu+3b+xzl65cQAYsUeViN54gjB474UdfOL/iaFLBIkYcl6+/t200cMmSlv0kBiwRYTv5+huUEWCTAkqtvOnfAZY+ZvpN2BFgkwJrSSXQ5SIg+231NClikyMN6p6DfiiZZPrXrRb4mBSxS5GFdd1jLnt/XaaefSQGLFHlYAwa3vF8r3vEzKWCRIg9r2Mkt7xfiJ+8ZAVZO6bD+IqY47/7T5wjT5noGWKTIw2o4XXx75G+G7l/wD1+TAhYp8rBk1ZhDhej0/Zf8TQpYJMCyr+iOLX5/MxmwaJGGVVfjDvexaICVU+6Hf7nDw79ogJVTaVhTb3S30c+kgEWKNCzGAIsEWDwBFgmweAIsEmDxBFgkwOIJsEiRhnXdYCnf+jfLpIBFijSsc/tI2d/fQ+tbAyxSpGH9Qlz/wHFnTdndJ34mBSxSpGF9/HX85N10MmDllOvOe/O29d+9umR3vv4hGrBI0YZlN3Uuy6SARYo8LDyuMHuAlVN4XKEKsDjD4wpVgMUZHleoAizO8LhCFWBxhscVqgCLMzyuUAVYnOFxhSrA4gyPK1QBFmd4XKEKsDjDL/qpAIszwFIBFmeApQIszgBLBVicAZYKsDijsCrWrV7lVOlnUsAiRR5Ww/Vd8avJWQKsnNJhPSwGTnoo1ad+JgUsUuRhDbqQZVLAIkUe1sDbMz/j7XvGLyHHzS/dO2GuYY8AixQ0WB8W+/rtlfakw7qjf8YnLB769DND5+rHf7rs7wuuuT37SxoCFilIsF6/X8rnxbOen+czHVbiuBtWx8qcWneg6YoFUv7rkjr3caJotZSfFK3POilgkYIEq2CklNumbfb8PJ/psAZ3pw9Y3Vy00141R1L62Hqq3v4L5KDXsk4KWKQgwRIjPT+FI/K3wjGtbdp9yVsXObd4l75Aj+WSQeqRPI0NpCqril7kqt4qM4w2JK06w2iVvo50OJYwTV1u1RpGa6wK08nxuGm00qo2jNZZ5aaTyyzTqNdq/jdjRQz7vaNY9C3+JHUfq+6f9z2w1NnR1oN1xR/aH20tXmW/TT4x4RFfPxnw+sn7vy523l49jx5vu3SG+pyktSfTlnGPfuWA9t+MFTG8oPemU0SPUz5w7mMl+4rDDxJnVKcPnhXP25/xuphp35s+RBzWqesjjLDoA1aXDHPeXrGIHH90+fj0C7FWV5KSVpJe5M6Km0bjVoVhNKmvIx2OGadOWOWG0XKrzHRyLGYaLTN+yxVWwnRy3DKNmlezgsByLjO+UrxzU+jAGlv4tmy+T0xPH6RhbTvg5E0yOazTG2ywMh6wuqrIvu/QPOQN/XjNJZNNL/CL+1ikoN3HcmBd0eUj+57y7A/TB2lYt4m19tGuLw73nC37l9E+ynzAanJQiX2zW1SqHa8Z8ojxDwVgkYIIa1HB/kMe/Y99gTpIwzqrMPWc7D2O8Zwt+5fRPmrjAat33lpbf+ctUm7ZoI4rRkyts8u+/4BFCiIs+fyAzqLgvG3pgzSs73Trl2qg52zZv4z2URsPWLVuHDb8hh1STrxJHS8oSvVi1kkBixRIWFJWzLtMnJM+mCMWSmdspjz9C74e/pf6MtpHbT1gtXnzx85X2fqhOi5dkyqRdVLAIgUR1q+udD76waHpgwXir/bBXTas3wjnZ0q7TvqN52zZv4z2ER6wmr0AwdpHK0dY3U7fWe/AGl3wVymXH3Be+qB035O3y0UH27A2dj1qhaz8qZjtOVvW8IBVVURgnSXEXAdWxWniwMPF1zanD+RYUdCty4POz7GeO1Acul/hLZ6TZQ8PWFVFBFZy+qR1qZ+8N7/84D3znWutDuS7k/+4uSb1k/fE7AmPbvCcyxAesKqKCKw9VBpW0mp0/eNMvZ9JAYsUaVj9RYnrxTDxO+80wMqpNKwZY3fa/7Xm6/d1AIsUaVhan+GmkAZYOUV+8r77L5hzxCo/kwIWyQuW0UbYYdXNnjXr6wNnOf3fBWKFn0kBixRpWPIy10vp9DX9VoxngEWKNqyyJUtOuniJ09JVvu5iARYt2rDs/uHjVwZdARap0XytOz4se/k2lpSUrFsxo8TPpIBFijysuksL8QPSLHUUWPtkzfPUXNJhTRdn/6HL5b/vK6bhzjsNsHJKhzX82AbZ7xbZOOi7viYFLFLkYZ1zsZS/GCDlcoEXaaIBVk7psIYOlPLenlJaYrmfSQGLFHlYk7sul4vEMrlA4G+FNMDKKfJsM70Knq3vffBFBxxj2gHPAIsUeVhy2w3vyPePE8f5erlCwKIBVqrmcp+TAhap48IafXaWAddvN9S4w8O/aIDVVu2A1V+4w0/eaZGCtbmkafV7qX2IL9tqv920vmJVjX3hyjXNcvuy1C3ap2+uaWwXrKk3utvoeQUNARYpdLDGntXn1INP2C7rry/8SqfBu+SYs3sUPjz27FO+d+DZE4454UvvyuS5XY/ucuy29sBy2uF5tdoVYJHCB0sslskj7pS3H/Se3Nj7ZjlGLKquGlvwpnxbnNsgz/up/N3XYnJn71vbC+vMk+//zPOaeQdYpPDBOt5+c/EIefRt9vuHD5JjejsXfsNef/GElLf/QH60wb7aZ1zdXliPnyI6nf2Y378UAhYtfLDOsN+MuER2etJ+v1TEx/RzLjxdygbnqY6KT5O1959z5BcPvrK9sKTccMcJouvw50yr5B1gkcIHy3nCfxvWYdPt9/NF9Zj+uy9shTWy9+Nr6wflAMtu5W19Cw667m3Pa5g9wCKFFtaF59nvr/6mzIDV8x77e/7y5TnBKp1+4X4FXxKnrfW8jtkCLFJoYa3pNvyxkV0WZcIa+NUnZp22/+D2w4o/8qNO4sS7tzTOP/ww4zqbAixS6GDNGG2/uetOKdf+YsC179n330fvvrCx/zIpZ14v478779KZc4fKh7I9N5sOa97AzuKwX7c8pvAB8a7nlcwSYJFCB8t/Oqz+XxjxfOsvJT89xLhJpgCLFHlYr33uWz8twCL5g5UxGj5YzmtCr1juZHyApVeARYo8rMZr9sU/QmcJsHJKhzVNnHr3tFS+/mUHsEiRh3XRUSYR7Q6wSJGHdYGP17hwBVikIMHaQ+mw7jusmmNSwCJFGlZlWVlZ6bfPeXOH9prQnyvAIkUaFn41ORqw/ps1z1NzKQ0r/XrQ7teE/lwBFinSsLTwrMkZAVZO4VmTVYDFGZ41WQVYnOFZk1WAxRmeNVkFWJzhWZNVgMUZ/VvhpvEjnpQPfu5f8WsJsEihg1VyjOc86dr8vXcCa9q+Qty2Qxzp68U1AYsWOlj/LPScJ107YC0rGLLuxNuanjzwjBzmzQywSGGDtfVnBcUfPv+4ffTUPLl47kf3P/ifpjmTltofNzw36fFKKV+c98aUtbJ89t0PbmgXrOuOrpV9bpNylsDvY9GiBGvLsIKxG1JefjJSju3znVtO7TXwZ9cVPiFLv9X70q/13iLHnNjr+3PX9/yfq/p2fqc9sAZcIlOw1gs/j1cFLFrYYKVuChWs/UrlZjFKypFF8ooTymXN9y+SYzpvlnLUmc1SfmdUe2BdcloLrGeErztZgEXKhJWDjb0O62R7bcTfpfzf78ke99uXPd2teYzzpCGyqemjeX2uag+s2eIRB9b7R3zV8/qZAixSuGG5HgBd+LR92evCGnOG/X5bUZcePz4hy/M36LCaLhB9Dvj2Dws7v+R5/UwBFimUsG5yHlbfX4d19AP2ZU8X1qcecv+jflulPLddsGTDvV8Xotv5yzyvnjHAIoUO1gsFjXLi4dVyUzcd1i9P3CXrTh/Q8lwOPf8g5aYvXNYuWHZ1SV9PbOsEWKTQwdpY2Oelj7sfP7jXmTos6/hjrvjGURtbYI3e/9qRPb5xbrtgNS2f81QqX08aCVik0MGSr0xYIz/945StL86Ti2faLoo/lHLJo/aVefLuJ+xlenGm80mL7nnkw1UTZeoHXjQd1s5v41eTsxUtWL7TYd3c7b5Fi1MlcpijIkGKW3F6kTvLMo3GjKNxfYsypo6ZpzZdr7jHycZR87ecea1NNujJGaNeJzebdmuvwDr/JpZJ8X8sUuT/j/XzUSyTAhYp8rBWHOTv1Zl2B1ikyMOSNxScOOBcp/f9TApYpMjDmisKevRK9ZafSQGLFHlYw/p8wjEpYJGCBGsPpcM673aWSQGLFHlYt17AMilgkSIPyzrqune349lm2gywckqHNfgA/JNOtgArp3RYD+PZZrIGWDmV5dlm/AVYJMDiCbBIgMUTYJEAiyfAIgEWT4BFAiyeAIsEWLJi3epVTpV+JgUsUuRhNVzfFT8gzRJg5RT5AakYOOmhVJ/6mRSwSJGHNehClkkBixR5WAPxazNZA6yc0mHd0Z9lUsAiRR5W4rgbVsfwazNtBlg5RX5tpjt+bSZbgJVT+LUZFWBxhp+8qwCLMworsbGkpGTdihklfiYFLFLkYdVdWoj7WFkCrJzSYU0XZ/+hy+W/7yum4UWaaICVUzqs4cc2yH63yMZB3/U1KWCRIg/rnIul/MUAKZeLnX4mBSxS5GENHSjlvT2ltMRyP5MCFinysCZ3XS4XiWVygcDfCmmAlVPkn3R6FTxb3/vgiw44Bv+kQwOsnCI/x9p2wzvy/ePEcf6efw2wSICVqrnc56SARQIsvMJq1gArp/AKqyrA4gyvsKoCLM7wCqsqwOIMr7CqAizO8AqrKsDiDK+wqgIszvAKqyrA4gyvsKoCLM7wCqsqwOIsDav0Y3c1fiYFLFKkYfUX7vA77zTAyik3rG4XjcHjCrMGWDmVhrX0mkPFd8b/m2NSwCJFGpb97b8yqpc48fbVvicFLFLEYdk1v33zseKrY9/1NylgkQDLadW4b4oj1/qZFLBIgOW0ZeqPCsVrfiYFLBJgyffv7CvEKXf5uw8PWKSIw2p6/TfHisIzpmz1OylgkSINa8nInqLL+Y/6egj07gCLFGlY/cXBV/zt+daMq+gVYJEiDgv/pGM6GbByKg1rwaPu/uNnUsAiRRoWY4BFAiyeAIsEWG3UvG1zc+axZfrbI2CRACuzxOjhw39ZSo+rRz1rOAewSICV2fhxdfV33EKOEzcXAZYWYJG8YCUHrZVyS9Fn2vGaS4qvASwtwCJ5wXqvqMamMOQ17Xjl6803ApYWYJG8YL061Hl7xQJ67IZVs4tUbpXTi9xZcdNowqo0jJbrW0SHY8apy6wKw2iFVWY6ORYzjSaN33JFxrU22aBTZ4x6ndycuY97PC9Y/7rYeXv1fHrshpW09mTaMu7Rr+wneq1NNozntufkMMB64yLnWl66mB67YTU2kKqsKnqRq3qrzDDakLTqDKNV+jLT4VjCNHW5VWsYrbEqTCfH46bRSqvaMFqXca1NNsjJVRmjXifnyUpOecHaXBSz70AUfUCPcR9LD/exSF6wmi5bKOXLl9TSY8DSAyyS58+xFg158m/DnpNy9oPpYwlYNMAief+Tzmt3TXjVfvfEn9LHdlNfNZwCWCTA4gmwSIDFUxRhGbcXsHgCLDIKWDwBFhkFLJ4Ai4wCFk+ARUYBiyfAIqOAxRNgkVHA4gmwyChg8QRYZBSweAIsMgpYPAEWGQUsngCLjAIWT4BFRgGLJ8Aio4DFkz9YxoUCrNxO3msBlgqwOAsGrPYvFGDldvJeC7BUgMUZYKkAizPAUgEWZ4ClAizOAEsFWJwBlgqwOAsDLMM60on8wTJODVg5FXFYOUwNWDm1V2BlrARgAVZ7ylwKd4Bl/sKAlbW9CMt8sp+pASunAAuw8hJgAVZeAizAykuABVh5CbAAKy8BFmDlJcACrLwEWIap9VHAyqXww/JYZcDaO7HBav/3Clje19odYAFWtm8ZsD5/HR+W8ZsCLBJgAVZeAqx8Te0OsD5/fjYQsEwnAxZg7f2pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pwYsbSWCuEXhnBqwtJUI4haFc2rA0lYiiFsUzqkBS1uJIG5ROKcGLG0lgrhF4ZwasLSVCOIWhXNqwNJWIohbFM6pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pwYsbSWCuEXhnLpDwWqotzN8r3R0Hzrq62TTKkdv6tRyMuyp7zhgVVfaGb5XOroPHfV1smmVozd1ajmbGTbVb7gp7FhTd6ibwlR+VjmQWxTOqQFLW4kgblE4pwYsbSWCuEXhnBqwtJUI4haFc2rA0lYiiFsUzqkBS1uJIG5ROKcGLG0lgrhF4ZwasLSVCOIWhXNqwNJWIohbFM6pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pwYsbSWCuEXhnBqwtJUI4haFc2rA0lYiiFsUzqkBS1uJIG5ROKcGLG0lgrhF4ZwasLSVCOIWhXNqwNJWIohbFM6pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pwYsbSWCuEXhnBqwtJUI4haFc2rA0lYiiFsUzqkBS1uJIG5ROKcGLG0lgrhF4ZwasLSVCOIWhXNqwNJWIohbFM6pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pwYsbSWCuEXhnBqwtJUI4haFc2rA0lYiiFsUzqkBS1uJIG5ROKcGLG0lgrhF4ZwasLSVCOIWhXNqwNJWIohbFM6pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pwYsbSWCuEXhnBqwtJUI4haFc2rA0lYiiFsUzqkBS1uJIG5ROKcGLG0lgrhF4ZwasLSVCOIWhXNqwNJWIohbFM6pAUtbiSBuUTinBixtJYK4ReGcGrC0lQjiFoVzasDSViKIWxTOqQFLW4kgblE4pw4RrNjC50rpsfuy3flZ5UBuUTinDg+s9cPHjRu6Uj92X9aan1UO5BaFc+rwwBo9Q8qZ1zRrx+7LWvOzyoHconBOHRpYpUVb7Fu+oo3uY/dlKj+rHMgtCufUoYG1bFCj/Xb4y+5j92UqP6scyC0K59ShgfXKcOftVfPdx+7LnCoSdobvlY7uQ0d9nWxa5ehNnVrO5rZ2cg/nBevlFKIrF7iP3Zc5pWC5i1txepE7yzKNxoyjXlPHzFObTo57nGwcNV8vr6k9vmXjyW2MhgFWy83exa+4j92XtVmNVWuYstlKmr5iudVkGK21qk0nx8pMo5VWg2G03tplOjkRN41WWXWG0UarwnRy0jKN1lg1hlGP1dxred95/0TKeNFm97H7sjYDLBJgZTZqupSPX9skm5vSx63vswVYJMDKbO3Q398+ZLWUE29KH7e+zxZgkQCrjXbMm7/dfvf6C+lj9T5LgEUCLJ4AiwRYPAEWCbB4AiwSYPEEWCTA4gmwSIDFE2CRAIsnwCIBFk+ARQIsngCLBFg8ARYJsHgCLBJg8QRYJMDiCbBIgMUTYJEAiyfAIgEWT4BFAiyeAIsEWDwBFgmweGqsaTQN15j2QNbVmB4W11hjsiFrTKJlvXHqJvPUtcapG2pMfxyaa+pNJ9eZ5Phbzb1WXmAhBFgoLwEWykuAhfISYKG8BFgoLwEWykuAhfISYKG8BFgoLwEWykuAhfISYKG8BFgoLwEWykuAhfISYKG8BFgoLwEWykuAhfISYKG8BFgoLwEWykuAhfISYKG8BFgoL3UwWB8U291xz6zSzKHFxVUeJ79fvLDl4Kk7DZ/1YrHxAfGopQ4G6ynR0n6Pui9t/ostZqxo+6k3UoOpZonun6UOflJo+BJjhPGJR1BLHQ7WpNLS0o/nHLLvBtelS8UsKd+Z1vb/aVKDqWYJMTR1AFj+63CwpqXezxT/67p0ibLTRktcsA4R850DwPJfB4W1UYyw3773p7seLpFy2ZVi8Pjd97G2TJ/wtPM0WC9OlkvvnaoGU80SD33xSGfQgbWw2HlSo6XFZfanNr90z58/lZWzJjk3mjasf038S8vNautsz9+/6+EHSvb4dxvkOiisN8V1srpIfOFQUTBZPnO8OOYHLfexJnXet6f48jKbR49hB/6wZ6fpuwdTzRJzp4ibZAusG4VzyzlefCzHHHLhoT/cv/uzvfucJC53YA0VPcSBzisZq9lGf/lMYfq/YgTrmLAah4mn5d3ij01y0/EHVLfc2jmwnhOXV8gNfXombR5nlcnKb/Ws124K5zaeUrgyA5boX2XfExMPS3mD2GB/2Os9uab3wQnXbKPFGds3eP2lM1p1OFgXFBcXX3+iOLVBPjrKuWSc2JSG1e8rztPfLRV/tnm8ah/daUPRYMmVhac0ZsBabB91P9J+M8++DzbGNivlE7bg9GyjxUt7/nsNdh0OVqqev0w4H1W++/S4r4v1ClZj5yNvtLtWXG3z2G5/woNiNYElfy2mZMD61D7qdab9ZrF4yv6w0j6yxM9ds40WbfzgLNp1OFiTSku3tzyTbHJEoej6vX6iRMFKijsG1jIAAAGnSURBVMP7pbpt99/tHhKrKKzKI7+4jcJyPrXXuXI3rANSn7zvcNdso1PYkKsOB2uaOv5x5wnrGuUdLlgNnc5rHcwKS84XgxxYo4Rzn2lsG7AKnWfILRO/dM0GWBl1XFgV4gLn3YXi3/LV1vtYJ3ffaV/01omzXbBeJbDkEHGwDetmscU+PqcNWOI9+2ih+KtrNsDKqOPCaj7w6JisnyxsO8vEb2MpWLPE2dvllm/ut9kFKzWYajesT7sLG9Yc8fOausmFbcEaWC/jJx1R5ZoNsDLquLDkI4X7f/NL3a4Qc2TyS0KUpX6ONa6w4Muduv/dfVOYGkydsBuWfZfehtXwQ9G582F3tAFr/+8d+5NDD18hXbMBVkYdDNYHxcvTH5Q8fO9TO6qcX1nYdN8DZS0/ef9o+sTZzktMtPyOwrLi0pbB1Oe/X9zyw/Ome5zfbmj658THykqcn7ynPvV+5wZzc/EH8sWJDXPufrzl7wetsz1fHMxn8d+LdTBYKCgBFspLgIXyEmChvARYKC8BFspLgIXyEmChvARYKC8BFspLgIXyEmChvPT/euS37/VfFeEAAAAASUVORK5CYII=",
      "text/plain": [
       "plot without title"
      ]
     },
     "metadata": {
      "image/png": {
       "height": 300,
       "width": 300
      }
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "df <- sample_metadata\n",
    "df$mean_methylation <- df$mean_methylation / 100\n",
    "df$patient_number <- factor(df$patient_number)\n",
    "p <- ggplot(df, aes(x = patient_number, y = mean_methylation, fill = tissue)) +\n",
    "  geom_bar(stat = \"identity\", position = \"dodge\") +\n",
    "  xlab(\"Patient Number\") +\n",
    "  ylab(\"Mean methylation of loci\") +\n",
    "  scale_x_discrete(labels = function(x) vector(\"character\", length(x))) + # Hide x-axis labels\n",
    "#   ggtitle(\"Proportions of Methylation Values\\nBelow 0.2 and Above 0.8\") +\n",
    "  theme_minimal() +\n",
    "  scale_fill_manual(values = c(\"normal\" = \"blue\", \"tumoral\" = \"red\"))\n",
    "\n",
    "ggsave(\n",
    "  p,\n",
    "  file = file.path(\n",
    "    outdir,\n",
    "    paste0(\n",
    "        \"mean_methylation_\",\n",
    "        context,\n",
    "        \".pdf\"\n",
    "    )\n",
    "  ),\n",
    "  width = 5,\n",
    "  height = 5\n",
    ")\n",
    "p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3e4fcfb5",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 4.3.2 (2023-10-31)\n",
       "Platform: x86_64-conda-linux-gnu (64-bit)\n",
       "Running under: Pop!_OS 22.04 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS/LAPACK: /home/gcharbonnier/mambaforge/envs/rkernel_diff_meth_v2/lib/libopenblasp-r0.3.24.so;  LAPACK version 3.11.0\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_DK.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_DK.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_DK.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_DK.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "time zone: Europe/Paris\n",
       "tzcode source: system (glibc)\n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats4    stats     graphics  grDevices utils     datasets \n",
       "[8] methods   base     \n",
       "\n",
       "other attached packages:\n",
       " [1] reshape2_1.4.4                         \n",
       " [2] HDF5Array_1.28.1                       \n",
       " [3] rhdf5_2.44.0                           \n",
       " [4] DelayedArray_0.26.6                    \n",
       " [5] S4Arrays_1.0.4                         \n",
       " [6] tidyr_1.3.0                            \n",
       " [7] ggpubr_0.6.0                           \n",
       " [8] ggplot2_3.4.4                          \n",
       " [9] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2\n",
       "[10] GenomicFeatures_1.52.1                 \n",
       "[11] org.Hs.eg.db_3.18.0                    \n",
       "[12] AnnotationDbi_1.64.1                   \n",
       "[13] irlba_2.3.5.1                          \n",
       "[14] Matrix_1.6-1.1                         \n",
       "[15] ChIPpeakAnno_3.36.0                    \n",
       "[16] DSS_2.48.0                             \n",
       "[17] bsseq_1.38.0                           \n",
       "[18] SummarizedExperiment_1.30.2            \n",
       "[19] MatrixGenerics_1.12.2                  \n",
       "[20] matrixStats_1.0.0                      \n",
       "[21] GenomicRanges_1.52.0                   \n",
       "[22] GenomeInfoDb_1.36.1                    \n",
       "[23] IRanges_2.34.1                         \n",
       "[24] S4Vectors_0.38.1                       \n",
       "[25] BiocParallel_1.34.2                    \n",
       "[26] Biobase_2.60.0                         \n",
       "[27] BiocGenerics_0.46.0                    \n",
       "[28] openxlsx_4.2.5.2                       \n",
       "[29] BiocManager_1.30.22                    \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       "  [1] jsonlite_1.8.7            magrittr_2.0.3           \n",
       "  [3] farver_2.1.1              BiocIO_1.10.0            \n",
       "  [5] zlibbioc_1.46.0           vctrs_0.6.4              \n",
       "  [7] multtest_2.58.0           memoise_2.0.1            \n",
       "  [9] Rsamtools_2.16.0          DelayedMatrixStats_1.22.1\n",
       " [11] RCurl_1.98-1.13           base64enc_0.1-3          \n",
       " [13] rstatix_0.7.2             htmltools_0.5.7          \n",
       " [15] progress_1.2.2            lambda.r_1.2.4           \n",
       " [17] curl_5.1.0                broom_1.0.5              \n",
       " [19] Rhdf5lib_1.22.0           plyr_1.8.9               \n",
       " [21] futile.options_1.0.1      cachem_1.0.8             \n",
       " [23] uuid_1.1-1                GenomicAlignments_1.36.0 \n",
       " [25] lifecycle_1.0.3           pkgconfig_2.0.3          \n",
       " [27] R6_2.5.1                  fastmap_1.1.1            \n",
       " [29] GenomeInfoDbData_1.2.10   digest_0.6.33            \n",
       " [31] colorspace_2.1-0          regioneR_1.34.0          \n",
       " [33] RSQLite_2.3.3             labeling_0.4.3           \n",
       " [35] filelock_1.0.2            fansi_1.0.5              \n",
       " [37] abind_1.4-5               httr_1.4.7               \n",
       " [39] compiler_4.3.2            withr_2.5.2              \n",
       " [41] bit64_4.0.5               backports_1.4.1          \n",
       " [43] carData_3.0-5             DBI_1.1.3                \n",
       " [45] R.utils_2.12.2            ggsignif_0.6.4           \n",
       " [47] biomaRt_2.56.1            MASS_7.3-60              \n",
       " [49] rappdirs_0.3.3            rjson_0.2.21             \n",
       " [51] gtools_3.9.4              permute_0.9-7            \n",
       " [53] tools_4.3.2               zip_2.3.0                \n",
       " [55] R.oo_1.25.0               glue_1.6.2               \n",
       " [57] VennDiagram_1.7.3         restfulr_0.0.15          \n",
       " [59] InteractionSet_1.30.0     rhdf5filters_1.12.1      \n",
       " [61] grid_4.3.2                pbdZMQ_0.3-10            \n",
       " [63] generics_0.1.3            gtable_0.3.4             \n",
       " [65] BSgenome_1.68.0           R.methodsS3_1.8.2        \n",
       " [67] data.table_1.14.8         hms_1.1.3                \n",
       " [69] car_3.1-2                 xml2_1.3.5               \n",
       " [71] utf8_1.2.4                XVector_0.40.0           \n",
       " [73] pillar_1.9.0              stringr_1.5.0            \n",
       " [75] IRdisplay_1.1             limma_3.56.2             \n",
       " [77] splines_4.3.2             dplyr_1.1.3              \n",
       " [79] BiocFileCache_2.8.0       lattice_0.22-5           \n",
       " [81] survival_3.5-7            rtracklayer_1.60.0       \n",
       " [83] bit_4.0.5                 tidyselect_1.2.0         \n",
       " [85] RBGL_1.78.0               locfit_1.5-9.8           \n",
       " [87] Biostrings_2.68.1         futile.logger_1.4.3      \n",
       " [89] stringi_1.7.12            yaml_2.3.7               \n",
       " [91] evaluate_0.23             codetools_0.2-19         \n",
       " [93] tibble_3.2.1              graph_1.80.0             \n",
       " [95] cli_3.6.1                 IRkernel_1.3.2           \n",
       " [97] repr_1.1.6                munsell_0.5.0            \n",
       " [99] Rcpp_1.0.11               dbplyr_2.4.0             \n",
       "[101] png_0.1-8                 XML_3.99-0.15            \n",
       "[103] blob_1.2.4                prettyunits_1.2.0        \n",
       "[105] sparseMatrixStats_1.12.2  bitops_1.0-7             \n",
       "[107] scales_1.2.1              purrr_1.0.2              \n",
       "[109] crayon_1.5.2              rlang_1.1.2              \n",
       "[111] KEGGREST_1.40.0           formatR_1.14             "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2841aa8b",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "4.3.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
