{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "a2a6bcde",
   "metadata": {
    "slideshow": {
     "slide_type": "skip"
    },
    "tags": [
     "snakemake-job-properties",
     "remove-input"
    ],
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "\n",
    "######## snakemake preamble start (automatically inserted, do not edit) ########\n",
    "library(methods)\n",
    "Snakemake <- setClass(\n",
    "    \"Snakemake\",\n",
    "    slots = c(\n",
    "        input = \"list\",\n",
    "        output = \"list\",\n",
    "        params = \"list\",\n",
    "        wildcards = \"list\",\n",
    "        threads = \"numeric\",\n",
    "        log = \"list\",\n",
    "        resources = \"list\",\n",
    "        config = \"list\",\n",
    "        rule = \"character\",\n",
    "        bench_iteration = \"numeric\",\n",
    "        scriptdir = \"character\",\n",
    "        source = \"function\"\n",
    "    )\n",
    ")\n",
    "snakemake <- Snakemake(\n",
    "    input = list(),\n",
    "    output = list(),\n",
    "    params = list(),\n",
    "    wildcards = list(),\n",
    "    threads = 1,\n",
    "    log = list( 'rds' = 'out/ipynb/compute_unsupervised_classification/pca.rds'),\n",
    "    resources = list('tmpdir', \"tmpdir\" = '/tmp'),\n",
    "    config = list(),\n",
    "    rule = 'ipynb',\n",
    "    bench_iteration = as.numeric(NA),\n",
    "    scriptdir = '/mnt/thymus/synoSalva/illumina_sequencing_data/mw/mw-oncodiag/src/snakemake/rules/../../ipynb',\n",
    "    source = function(...){\n",
    "        wd <- getwd()\n",
    "        setwd(snakemake@scriptdir)\n",
    "        source(...)\n",
    "        setwd(wd)\n",
    "    }\n",
    ")\n",
    "# setwd('/mnt/thymus/synoSalva/illumina_sequencing_data/mw/mw-oncodiag');\n",
    "setwd('/srv/shiny-server/mw-oncodiag');\n",
    "\n",
    "######## snakemake preamble end #########\n",
    "context <- \"CpG\"\n",
    "# context <- \"CHH\"\n",
    "# context <- \"CHG\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "8e449f62",
   "metadata": {
    "slideshow": {
     "slide_type": "skip"
    },
    "tags": [
     "remove-input"
    ],
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# start coding here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "7690591a-7450-4f4f-9f1d-1b9540939f47",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/markdown": []
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "IRdisplay::display_markdown(\n",
    "    sprintf(\n",
    "\"_Compute unsupervised classification %s_\n",
    "\n",
    "__Oncodiag project__\n",
    "\n",
    "[Guillaume Charbonnier](mailto:gc.bioinfo@gmail.com)\n",
    "\n",
    "%s\",\n",
    "        snakemake@wildcards$design,\n",
    "        format(Sys.Date(), \"%Y-%m-%d\")\n",
    "    )\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "082c2ea6-9d05-48c3-9cf2-ee88852b2844",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    },
    "tags": []
   },
   "source": [
    "# Abstract\n",
    "\n",
    "The aim of this analysis is to assess the main components of variance in the dataset."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b43c03ed-8ab7-482b-8797-bf9c310f6db1",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    },
    "tags": []
   },
   "source": [
    "# Methods\n",
    "\n",
    "## Load depencies\n",
    "\n",
    "### Packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "8a4fe98c-a611-4d31-8303-f02876ccc4cd",
   "metadata": {
    "tags": [],
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'getOption(\"repos\")' replaces Bioconductor standard repositories, see\n",
      "'help(\"repositories\", package = \"BiocManager\")' for details.\n",
      "Replacement repositories:\n",
      "    CRAN: https://cran.r-project.org\n",
      "\n",
      "Bioconductor version 3.18 (BiocManager 1.30.22), R 4.3.2 (2023-10-31)\n",
      "\n",
      "Warning message:\n",
      "“package(s) not installed when version(s) same as or greater than current; use\n",
      "  `force = TRUE` to re-install: 'data.table' 'openxlsx' 'DSS' 'bsseq'\n",
      "  'ChIPpeakAnno' 'irlba' 'org.Hs.eg.db' 'TxDb.Hsapiens.UCSC.hg19.knownGene'\n",
      "  'ggpubr' 'tidyr'”\n",
      "Loading required package: Biobase\n",
      "\n",
      "Loading required package: BiocGenerics\n",
      "\n",
      "\n",
      "Attaching package: ‘BiocGenerics’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    IQR, mad, sd, var, xtabs\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    anyDuplicated, aperm, append, as.data.frame, basename, cbind,\n",
      "    colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,\n",
      "    get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,\n",
      "    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,\n",
      "    Position, rank, rbind, Reduce, rownames, sapply, setdiff, sort,\n",
      "    table, tapply, union, unique, unsplit, which.max, which.min\n",
      "\n",
      "\n",
      "Welcome to Bioconductor\n",
      "\n",
      "    Vignettes contain introductory material; view with\n",
      "    'browseVignettes()'. To cite Bioconductor, see\n",
      "    'citation(\"Biobase\")', and for packages 'citation(\"pkgname\")'.\n",
      "\n",
      "\n",
      "Loading required package: BiocParallel\n",
      "\n",
      "Loading required package: bsseq\n",
      "\n",
      "Loading required package: GenomicRanges\n",
      "\n",
      "Loading required package: stats4\n",
      "\n",
      "Loading required package: S4Vectors\n",
      "\n",
      "\n",
      "Attaching package: ‘S4Vectors’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:data.table’:\n",
      "\n",
      "    first, second\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:utils’:\n",
      "\n",
      "    findMatches\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    expand.grid, I, unname\n",
      "\n",
      "\n",
      "Loading required package: IRanges\n",
      "\n",
      "\n",
      "Attaching package: ‘IRanges’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:data.table’:\n",
      "\n",
      "    shift\n",
      "\n",
      "\n",
      "Loading required package: GenomeInfoDb\n",
      "\n",
      "Loading required package: SummarizedExperiment\n",
      "\n",
      "Loading required package: MatrixGenerics\n",
      "\n",
      "Loading required package: matrixStats\n",
      "\n",
      "\n",
      "Attaching package: ‘matrixStats’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:Biobase’:\n",
      "\n",
      "    anyMissing, rowMedians\n",
      "\n",
      "\n",
      "\n",
      "Attaching package: ‘MatrixGenerics’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:matrixStats’:\n",
      "\n",
      "    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,\n",
      "    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,\n",
      "    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,\n",
      "    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,\n",
      "    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,\n",
      "    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,\n",
      "    colWeightedMeans, colWeightedMedians, colWeightedSds,\n",
      "    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,\n",
      "    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,\n",
      "    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,\n",
      "    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,\n",
      "    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,\n",
      "    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,\n",
      "    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,\n",
      "    rowWeightedSds, rowWeightedVars\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:Biobase’:\n",
      "\n",
      "    rowMedians\n",
      "\n",
      "\n",
      "Loading required package: parallel\n",
      "\n",
      "Loading required package: Matrix\n",
      "\n",
      "\n",
      "Attaching package: ‘Matrix’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:S4Vectors’:\n",
      "\n",
      "    expand\n",
      "\n",
      "\n",
      "Loading required package: AnnotationDbi\n",
      "\n",
      "\n",
      "\n",
      "Loading required package: GenomicFeatures\n",
      "\n",
      "Loading required package: ggplot2\n",
      "\n",
      "\n",
      "Attaching package: ‘tidyr’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:Matrix’:\n",
      "\n",
      "    expand, pack, unpack\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:S4Vectors’:\n",
      "\n",
      "    expand\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "if (!require(\"BiocManager\", quietly = TRUE)) {\n",
    "    install.packages(\n",
    "        \"BiocManager\",\n",
    "        quiet = TRUE\n",
    "    )\n",
    "}\n",
    "packages <- c(\n",
    "  \"data.table\",\n",
    "  # \"dplyr\",\n",
    "  \"openxlsx\",\n",
    "  # \"methylSig\",\n",
    "  \"DSS\",\n",
    "  \"bsseq\",\n",
    "  \"ChIPpeakAnno\",\n",
    "  \"irlba\",\n",
    "  # \"ChIPseeker\",\n",
    "  # \"methyAnalysis\",\n",
    "  \"org.Hs.eg.db\",\n",
    "  \"TxDb.Hsapiens.UCSC.hg19.knownGene\",\n",
    "  # \"EnhancedVolcano\",\n",
    "  # \"data.table\",\n",
    "  \"ggpubr\",\n",
    "  # \"ComplexHeatmap\",\n",
    "  \"tidyr\"\n",
    ")\n",
    "BiocManager::install(\n",
    "    packages,\n",
    "    update = FALSE,\n",
    "    quiet = TRUE,\n",
    "    Ncpus = parallel::detectCores()\n",
    ")\n",
    "invisible(\n",
    "    lapply(\n",
    "      packages,\n",
    "      library,\n",
    "      character.only = TRUE\n",
    "    )\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "2518be08-c1f3-4fb4-a1d8-0238a0c0e829",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 4.3.2 (2023-10-31)\n",
       "Platform: x86_64-conda-linux-gnu (64-bit)\n",
       "Running under: Pop!_OS 22.04 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS/LAPACK: /home/gcharbonnier/mambaforge/envs/rkernel_diff_meth_v2/lib/libopenblasp-r0.3.24.so;  LAPACK version 3.11.0\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_DK.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_DK.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_DK.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_DK.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "time zone: Europe/Paris\n",
       "tzcode source: system (glibc)\n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats4    stats     graphics  grDevices utils     datasets \n",
       "[8] methods   base     \n",
       "\n",
       "other attached packages:\n",
       " [1] tidyr_1.3.0                            \n",
       " [2] ggpubr_0.6.0                           \n",
       " [3] ggplot2_3.4.4                          \n",
       " [4] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2\n",
       " [5] GenomicFeatures_1.52.1                 \n",
       " [6] org.Hs.eg.db_3.18.0                    \n",
       " [7] AnnotationDbi_1.64.1                   \n",
       " [8] irlba_2.3.5.1                          \n",
       " [9] Matrix_1.6-1.1                         \n",
       "[10] ChIPpeakAnno_3.36.0                    \n",
       "[11] DSS_2.48.0                             \n",
       "[12] bsseq_1.38.0                           \n",
       "[13] SummarizedExperiment_1.30.2            \n",
       "[14] MatrixGenerics_1.12.2                  \n",
       "[15] matrixStats_1.0.0                      \n",
       "[16] GenomicRanges_1.52.0                   \n",
       "[17] GenomeInfoDb_1.36.1                    \n",
       "[18] IRanges_2.34.1                         \n",
       "[19] S4Vectors_0.38.1                       \n",
       "[20] BiocParallel_1.34.2                    \n",
       "[21] Biobase_2.60.0                         \n",
       "[22] BiocGenerics_0.46.0                    \n",
       "[23] openxlsx_4.2.5.2                       \n",
       "[24] data.table_1.14.8                      \n",
       "[25] BiocManager_1.30.22                    \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       "  [1] jsonlite_1.8.7            magrittr_2.0.3           \n",
       "  [3] BiocIO_1.10.0             zlibbioc_1.46.0          \n",
       "  [5] vctrs_0.6.4               multtest_2.58.0          \n",
       "  [7] memoise_2.0.1             Rsamtools_2.16.0         \n",
       "  [9] DelayedMatrixStats_1.22.1 RCurl_1.98-1.13          \n",
       " [11] base64enc_0.1-3           rstatix_0.7.2            \n",
       " [13] htmltools_0.5.7           S4Arrays_1.0.4           \n",
       " [15] progress_1.2.2            lambda.r_1.2.4           \n",
       " [17] curl_5.1.0                broom_1.0.5              \n",
       " [19] Rhdf5lib_1.22.0           rhdf5_2.44.0             \n",
       " [21] futile.options_1.0.1      cachem_1.0.8             \n",
       " [23] uuid_1.1-1                GenomicAlignments_1.36.0 \n",
       " [25] lifecycle_1.0.3           pkgconfig_2.0.3          \n",
       " [27] R6_2.5.1                  fastmap_1.1.1            \n",
       " [29] GenomeInfoDbData_1.2.10   digest_0.6.33            \n",
       " [31] colorspace_2.1-0          regioneR_1.34.0          \n",
       " [33] RSQLite_2.3.3             filelock_1.0.2           \n",
       " [35] fansi_1.0.5               abind_1.4-5              \n",
       " [37] httr_1.4.7                compiler_4.3.2           \n",
       " [39] withr_2.5.2               bit64_4.0.5              \n",
       " [41] backports_1.4.1           carData_3.0-5            \n",
       " [43] DBI_1.1.3                 HDF5Array_1.28.1         \n",
       " [45] R.utils_2.12.2            ggsignif_0.6.4           \n",
       " [47] biomaRt_2.56.1            MASS_7.3-60              \n",
       " [49] rappdirs_0.3.3            DelayedArray_0.26.6      \n",
       " [51] rjson_0.2.21              gtools_3.9.4             \n",
       " [53] permute_0.9-7             tools_4.3.2              \n",
       " [55] zip_2.3.0                 R.oo_1.25.0              \n",
       " [57] glue_1.6.2                VennDiagram_1.7.3        \n",
       " [59] restfulr_0.0.15           InteractionSet_1.30.0    \n",
       " [61] rhdf5filters_1.12.1       grid_4.3.2               \n",
       " [63] pbdZMQ_0.3-10             generics_0.1.3           \n",
       " [65] gtable_0.3.4              BSgenome_1.68.0          \n",
       " [67] R.methodsS3_1.8.2         hms_1.1.3                \n",
       " [69] car_3.1-2                 xml2_1.3.5               \n",
       " [71] utf8_1.2.4                XVector_0.40.0           \n",
       " [73] pillar_1.9.0              stringr_1.5.0            \n",
       " [75] IRdisplay_1.1             limma_3.56.2             \n",
       " [77] splines_4.3.2             dplyr_1.1.3              \n",
       " [79] BiocFileCache_2.8.0       lattice_0.22-5           \n",
       " [81] survival_3.5-7            rtracklayer_1.60.0       \n",
       " [83] bit_4.0.5                 tidyselect_1.2.0         \n",
       " [85] RBGL_1.78.0               locfit_1.5-9.8           \n",
       " [87] Biostrings_2.68.1         futile.logger_1.4.3      \n",
       " [89] stringi_1.7.12            yaml_2.3.7               \n",
       " [91] evaluate_0.23             codetools_0.2-19         \n",
       " [93] tibble_3.2.1              graph_1.80.0             \n",
       " [95] cli_3.6.1                 IRkernel_1.3.2           \n",
       " [97] repr_1.1.6                munsell_0.5.0            \n",
       " [99] Rcpp_1.0.11               dbplyr_2.4.0             \n",
       "[101] png_0.1-8                 XML_3.99-0.15            \n",
       "[103] blob_1.2.4                prettyunits_1.2.0        \n",
       "[105] sparseMatrixStats_1.12.2  bitops_1.0-7             \n",
       "[107] scales_1.2.1              purrr_1.0.2              \n",
       "[109] crayon_1.5.2              rlang_1.1.2              \n",
       "[111] KEGGREST_1.40.0           formatR_1.14             "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a85e2abd-3457-47d3-aae0-5f07a7545f13",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "set_plot_dim <- function(\n",
    "    width = 16,\n",
    "    height = 9\n",
    ") {\n",
    "    options(\n",
    "        #jupyter.plot_scale=1,\n",
    "        repr.plot.width = width,\n",
    "        repr.plot.height = height\n",
    "    )\n",
    "}\n",
    "set_plot_dim(5,5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "c6c772dd",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/latex": [
       "'/mnt/Data\\_2/mw-oncodiag'"
      ],
      "text/markdown": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/plain": [
       "[1] \"/mnt/Data_2/mw-oncodiag\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "getwd()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d25c731a-bb17-4f55-8a22-99b61e0d04f7",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    },
    "tags": []
   },
   "source": [
    "### Data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre class=language-r><code>new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = structure(\"samples\", package = \"Biobase\"), \n",
       "<span style=white-space:pre-wrap>    package = \"Biobase\", group = list(), valueClass = character(0), </span>\n",
       "<span style=white-space:pre-wrap>    signature = \"object\", default = NULL, skeleton = (function (object) </span>\n",
       "<span style=white-space:pre-wrap>    stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", </span>\n",
       "<span style=white-space:pre-wrap>        \"samples\"), domain = NA))(object))</span></code></pre>"
      ],
      "text/latex": [
       "\\begin{minted}{r}\n",
       "new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = structure(\"samples\", package = \"Biobase\"), \n",
       "    package = \"Biobase\", group = list(), valueClass = character(0), \n",
       "    signature = \"object\", default = NULL, skeleton = (function (object) \n",
       "    stop(gettextf(\"invalid call in method dispatch to '\\%s' (no default method)\", \n",
       "        \"samples\"), domain = NA))(object))\n",
       "\\end{minted}"
      ],
      "text/markdown": [
       "```r\n",
       "new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = structure(\"samples\", package = \"Biobase\"), \n",
       "    package = \"Biobase\", group = list(), valueClass = character(0), \n",
       "    signature = \"object\", default = NULL, skeleton = (function (object) \n",
       "    stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", \n",
       "        \"samples\"), domain = NA))(object))\n",
       "```"
      ],
      "text/plain": [
       "new(\"standardGeneric\", .Data = function (object) \n",
       "standardGeneric(\"samples\"), generic = \"samples\", package = \"Biobase\", \n",
       "    group = list(), valueClass = character(0), signature = \"object\", \n",
       "    default = NULL, skeleton = (function (object) \n",
       "    stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", \n",
       "        \"samples\"), domain = NA))(object))\n",
       "<bytecode: 0x559b92b5e280>\n",
       "<environment: 0x559b92b56e18>\n",
       "attr(,\"generic\")\n",
       "[1] \"samples\"\n",
       "attr(,\"generic\")attr(,\"package\")\n",
       "[1] \"Biobase\"\n",
       "attr(,\"package\")\n",
       "[1] \"Biobase\"\n",
       "attr(,\"group\")\n",
       "list()\n",
       "attr(,\"valueClass\")\n",
       "character(0)\n",
       "attr(,\"signature\")\n",
       "[1] \"object\"\n",
       "attr(,\"default\")\n",
       "`\\001NULL\\001`\n",
       "attr(,\"skeleton\")\n",
       "(function (object) \n",
       "stop(gettextf(\"invalid call in method dispatch to '%s' (no default method)\", \n",
       "    \"samples\"), domain = NA))(object)\n",
       "attr(,\"class\")\n",
       "[1] \"standardGeneric\"\n",
       "attr(,\"class\")attr(,\"package\")\n",
       "[1] \"methods\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "9dc0108b-320d-4f11-9d95-a2c64c472ec5",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 5</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>1</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>2</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>3</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>4</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>5</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>6</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>7</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>8</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>9</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>10</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>11</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>12</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>13</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>14</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>15</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>16</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>21</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>22</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>23</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>24</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>25</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>26</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>27</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>28</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td></tr>\n",
       "\t<tr><th scope=row>29</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td></tr>\n",
       "\t<tr><th scope=row>30</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 5\n",
       "\\begin{tabular}{r|lllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr>\\\\\n",
       "\\hline\n",
       "\t1 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal \\\\\n",
       "\t2 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral\\\\\n",
       "\t3 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal \\\\\n",
       "\t4 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral\\\\\n",
       "\t5 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal \\\\\n",
       "\t6 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral\\\\\n",
       "\t7 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal \\\\\n",
       "\t8 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral\\\\\n",
       "\t9 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal \\\\\n",
       "\t10 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral\\\\\n",
       "\t11 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal \\\\\n",
       "\t12 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral\\\\\n",
       "\t13 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal \\\\\n",
       "\t14 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral\\\\\n",
       "\t15 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal \\\\\n",
       "\t16 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral\\\\\n",
       "\t21 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal \\\\\n",
       "\t22 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral\\\\\n",
       "\t23 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal \\\\\n",
       "\t24 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral\\\\\n",
       "\t25 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal \\\\\n",
       "\t26 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral\\\\\n",
       "\t27 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal \\\\\n",
       "\t28 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral\\\\\n",
       "\t29 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal \\\\\n",
       "\t30 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 5\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|\n",
       "| 1 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  |\n",
       "| 2 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral |\n",
       "| 3 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  |\n",
       "| 4 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral |\n",
       "| 5 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  |\n",
       "| 6 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral |\n",
       "| 7 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  |\n",
       "| 8 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral |\n",
       "| 9 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  |\n",
       "| 10 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral |\n",
       "| 11 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  |\n",
       "| 12 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral |\n",
       "| 13 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  |\n",
       "| 14 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral |\n",
       "| 15 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  |\n",
       "| 16 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral |\n",
       "| 21 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  |\n",
       "| 22 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral |\n",
       "| 23 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  |\n",
       "| 24 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral |\n",
       "| 25 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  |\n",
       "| 26 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral |\n",
       "| 27 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  |\n",
       "| 28 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral |\n",
       "| 29 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  |\n",
       "| 30 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral |\n",
       "\n"
      ],
      "text/plain": [
       "   sample  fastq_1                                                      \n",
       "1  ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "2  ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "3  ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "4  ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "5  ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "6  ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "7  ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "8  ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "9  ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "10 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "11 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "12 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "13 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "14 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "15 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "16 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "21 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "22 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "23 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "24 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "25 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "26 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "27 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "28 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "29 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "30 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "   fastq_2                                                       patient_number\n",
       "1  inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz   1            \n",
       "2  inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz   1            \n",
       "3  inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz   2            \n",
       "4  inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz   2            \n",
       "5  inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz   3            \n",
       "6  inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz   3            \n",
       "7  inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz   4            \n",
       "8  inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz   4            \n",
       "9  inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz   5            \n",
       "10 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz  5            \n",
       "11 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz  6            \n",
       "12 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz  6            \n",
       "13 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz  7            \n",
       "14 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz  7            \n",
       "15 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz  8            \n",
       "16 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz  8            \n",
       "21 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz 11            \n",
       "22 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz 11            \n",
       "23 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz 12            \n",
       "24 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz 12            \n",
       "25 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz 13            \n",
       "26 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz 13            \n",
       "27 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz 14            \n",
       "28 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz 14            \n",
       "29 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz 15            \n",
       "30 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz 15            \n",
       "   tissue \n",
       "1  normal \n",
       "2  tumoral\n",
       "3  normal \n",
       "4  tumoral\n",
       "5  normal \n",
       "6  tumoral\n",
       "7  normal \n",
       "8  tumoral\n",
       "9  normal \n",
       "10 tumoral\n",
       "11 normal \n",
       "12 tumoral\n",
       "13 normal \n",
       "14 tumoral\n",
       "15 normal \n",
       "16 tumoral\n",
       "21 normal \n",
       "22 tumoral\n",
       "23 normal \n",
       "24 tumoral\n",
       "25 normal \n",
       "26 tumoral\n",
       "27 normal \n",
       "28 tumoral\n",
       "29 normal \n",
       "30 tumoral"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sample_metadata <- read.csv(\n",
    "  \"src/sample_sheets/all.csv\"\n",
    ")\n",
    "# Here I temporarily subset the sample sheets to already available samples\n",
    "suffix <- paste0(\".markdup.sorted_\", context, \".bedGraph\")\n",
    "# input_dir <- \"out/nextflow/nfcore_methylseq/methyldackel/\"\n",
    "input_dir <- \"out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/\"\n",
    "\n",
    "files <- list.files(\n",
    "    path = input_dir,\n",
    "    pattern = suffix\n",
    ")\n",
    "samples <- sub(\n",
    "    suffix,\n",
    "    \"\",\n",
    "    files\n",
    ")\n",
    "sample_metadata <- sample_metadata[sample_metadata$sample %in% samples,]\n",
    "# Remove patient 9 and 10 because they have low coverae\n",
    "sample_metadata <- sample_metadata[!sample_metadata$patient_number %in% c(9,10),]\n",
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "cea6afe8-a07b-486b-9ee9-a1e0d9683807",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": [
    "# This will be used to load files with read.bismarck()\n",
    "sample_metadata$bedGraph <- paste0(\n",
    "    # 'out/nextflow/nfcore_methylseq/methyldackel/',\n",
    "    # After 2023-11-06, I reprocessed methyldackel outside the nextflow pipeline\n",
    "    # to have the possibility to generate CHH and CHG bedGraph files.\n",
    "    'out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/',\n",
    "    sample_metadata$sample,\n",
    "    '.markdup.sorted_',\n",
    "    context,\n",
    "    '.bedGraph'\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "fa00d848-d751-4ae5-a27a-e57d2c567743",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 6</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th><th scope=col>bedGraph</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ODG_080</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_081</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_082</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_083</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_084</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_085</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_086</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_087</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_088</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_089</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_090</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_091</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_092</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_093</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_094</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_095</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_100</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_101</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_102</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_103</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_104</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_105</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_106</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_107</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_108</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_109</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 6\n",
       "\\begin{tabular}{r|llllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue & bedGraph\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\tODG\\_080 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_080.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_081 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_081.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_082 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_082.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_083 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_083.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_084 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_084.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_085 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_085.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_086 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_086.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_087 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_087.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_088 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_088.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_089 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_089.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_090 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_090.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_091 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_091.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_092 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_092.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_093 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_093.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_094 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_094.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_095 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_095.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_100 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_100.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_101 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_101.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_102 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_102.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_103 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_103.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_104 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_104.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_105 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_105.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_106 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_106.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_107 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_107.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_108 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_108.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_109 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_109.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 6\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; | bedGraph &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|\n",
       "| ODG_080 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_081 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_082 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_083 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_084 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_085 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_086 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_087 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_088 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_089 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_090 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_091 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_092 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_093 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_094 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_095 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_100 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_101 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_102 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_103 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_104 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_105 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_106 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_107 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_108 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_109 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph |\n",
       "\n"
      ],
      "text/plain": [
       "        sample  fastq_1                                                      \n",
       "ODG_080 ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "ODG_081 ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "ODG_082 ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "ODG_083 ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "ODG_084 ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "ODG_085 ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "ODG_086 ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "ODG_087 ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "ODG_088 ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "ODG_089 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "ODG_090 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "ODG_091 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "ODG_092 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "ODG_093 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "ODG_094 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "ODG_095 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "ODG_100 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "ODG_101 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "ODG_102 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "ODG_103 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "ODG_104 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "ODG_105 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "ODG_106 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "ODG_107 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "ODG_108 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "ODG_109 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "        fastq_2                                                      \n",
       "ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz \n",
       "ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz \n",
       "ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz \n",
       "ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz \n",
       "ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz \n",
       "ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz \n",
       "ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz \n",
       "ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz \n",
       "ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz \n",
       "ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz\n",
       "ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz\n",
       "ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz\n",
       "ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz\n",
       "ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz\n",
       "ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz\n",
       "ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz\n",
       "ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz\n",
       "ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz\n",
       "ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz\n",
       "ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz\n",
       "ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz\n",
       "ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz\n",
       "ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz\n",
       "ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz\n",
       "ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz\n",
       "ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz\n",
       "        patient_number tissue \n",
       "ODG_080  1             normal \n",
       "ODG_081  1             tumoral\n",
       "ODG_082  2             normal \n",
       "ODG_083  2             tumoral\n",
       "ODG_084  3             normal \n",
       "ODG_085  3             tumoral\n",
       "ODG_086  4             normal \n",
       "ODG_087  4             tumoral\n",
       "ODG_088  5             normal \n",
       "ODG_089  5             tumoral\n",
       "ODG_090  6             normal \n",
       "ODG_091  6             tumoral\n",
       "ODG_092  7             normal \n",
       "ODG_093  7             tumoral\n",
       "ODG_094  8             normal \n",
       "ODG_095  8             tumoral\n",
       "ODG_100 11             normal \n",
       "ODG_101 11             tumoral\n",
       "ODG_102 12             normal \n",
       "ODG_103 12             tumoral\n",
       "ODG_104 13             normal \n",
       "ODG_105 13             tumoral\n",
       "ODG_106 14             normal \n",
       "ODG_107 14             tumoral\n",
       "ODG_108 15             normal \n",
       "ODG_109 15             tumoral\n",
       "        bedGraph                                                                                                                                 \n",
       "ODG_080 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph\n",
       "ODG_081 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph\n",
       "ODG_082 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph\n",
       "ODG_083 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph\n",
       "ODG_084 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph\n",
       "ODG_085 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph\n",
       "ODG_086 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph\n",
       "ODG_087 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph\n",
       "ODG_088 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph\n",
       "ODG_089 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph\n",
       "ODG_090 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph\n",
       "ODG_091 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph\n",
       "ODG_092 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph\n",
       "ODG_093 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph\n",
       "ODG_094 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph\n",
       "ODG_095 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph\n",
       "ODG_100 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph\n",
       "ODG_101 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph\n",
       "ODG_102 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph\n",
       "ODG_103 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph\n",
       "ODG_104 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph\n",
       "ODG_105 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph\n",
       "ODG_106 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph\n",
       "ODG_107 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph\n",
       "ODG_108 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph\n",
       "ODG_109 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# row.names should be declared to be used as row.names of the BSseq object\n",
    "# because DMLtest() uses these row.namesto select samples\n",
    "row.names(sample_metadata) <- sample_metadata$sample\n",
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'out/ipynb/compute_unsupervised_classification/pca.rds'"
      ],
      "text/latex": [
       "'out/ipynb/compute\\_unsupervised\\_classification/pca.rds'"
      ],
      "text/markdown": [
       "'out/ipynb/compute_unsupervised_classification/pca.rds'"
      ],
      "text/plain": [
       "[1] \"out/ipynb/compute_unsupervised_classification/pca.rds\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    " snakemake@log$rds"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/latex": [
       "'/mnt/Data\\_2/mw-oncodiag'"
      ],
      "text/markdown": [
       "'/mnt/Data_2/mw-oncodiag'"
      ],
      "text/plain": [
       "[1] \"/mnt/Data_2/mw-oncodiag\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "getwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 6</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th><th scope=col>bedGraph</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ODG_080</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_081</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_082</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_083</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_084</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_085</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_086</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_087</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_088</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_089</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_090</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_091</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_092</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_093</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_094</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_095</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_100</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_101</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_102</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_103</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_104</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_105</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_106</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_107</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_108</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "\t<tr><th scope=row>ODG_109</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 6\n",
       "\\begin{tabular}{r|llllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue & bedGraph\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\tODG\\_080 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_080.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_081 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_081.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_082 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_082.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_083 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_083.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_084 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_084.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_085 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_085.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_086 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_086.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_087 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_087.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_088 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_088.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_089 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_089.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_090 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_090.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_091 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_091.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_092 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_092.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_093 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_093.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_094 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_094.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_095 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_095.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_100 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_100.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_101 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_101.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_102 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_102.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_103 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_103.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_104 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_104.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_105 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_105.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_106 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_106.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_107 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_107.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_108 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_108.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\tODG\\_109 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_109.markdup.sorted\\_CpG.bedGraph\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 6\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; | bedGraph &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|\n",
       "| ODG_080 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_081 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_082 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_083 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_084 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_085 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_086 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_087 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_088 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_089 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_090 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_091 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_092 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_093 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_094 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_095 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_100 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_101 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_102 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_103 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_104 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_105 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_106 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_107 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_108 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph |\n",
       "| ODG_109 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph |\n",
       "\n"
      ],
      "text/plain": [
       "        sample  fastq_1                                                      \n",
       "ODG_080 ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "ODG_081 ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "ODG_082 ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "ODG_083 ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "ODG_084 ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "ODG_085 ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "ODG_086 ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "ODG_087 ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "ODG_088 ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "ODG_089 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "ODG_090 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "ODG_091 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "ODG_092 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "ODG_093 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "ODG_094 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "ODG_095 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "ODG_100 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "ODG_101 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "ODG_102 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "ODG_103 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "ODG_104 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "ODG_105 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "ODG_106 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "ODG_107 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "ODG_108 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "ODG_109 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "        fastq_2                                                      \n",
       "ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz \n",
       "ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz \n",
       "ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz \n",
       "ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz \n",
       "ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz \n",
       "ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz \n",
       "ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz \n",
       "ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz \n",
       "ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz \n",
       "ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz\n",
       "ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz\n",
       "ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz\n",
       "ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz\n",
       "ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz\n",
       "ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz\n",
       "ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz\n",
       "ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz\n",
       "ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz\n",
       "ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz\n",
       "ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz\n",
       "ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz\n",
       "ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz\n",
       "ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz\n",
       "ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz\n",
       "ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz\n",
       "ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz\n",
       "        patient_number tissue \n",
       "ODG_080  1             normal \n",
       "ODG_081  1             tumoral\n",
       "ODG_082  2             normal \n",
       "ODG_083  2             tumoral\n",
       "ODG_084  3             normal \n",
       "ODG_085  3             tumoral\n",
       "ODG_086  4             normal \n",
       "ODG_087  4             tumoral\n",
       "ODG_088  5             normal \n",
       "ODG_089  5             tumoral\n",
       "ODG_090  6             normal \n",
       "ODG_091  6             tumoral\n",
       "ODG_092  7             normal \n",
       "ODG_093  7             tumoral\n",
       "ODG_094  8             normal \n",
       "ODG_095  8             tumoral\n",
       "ODG_100 11             normal \n",
       "ODG_101 11             tumoral\n",
       "ODG_102 12             normal \n",
       "ODG_103 12             tumoral\n",
       "ODG_104 13             normal \n",
       "ODG_105 13             tumoral\n",
       "ODG_106 14             normal \n",
       "ODG_107 14             tumoral\n",
       "ODG_108 15             normal \n",
       "ODG_109 15             tumoral\n",
       "        bedGraph                                                                                                                                 \n",
       "ODG_080 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph\n",
       "ODG_081 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph\n",
       "ODG_082 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph\n",
       "ODG_083 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph\n",
       "ODG_084 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph\n",
       "ODG_085 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph\n",
       "ODG_086 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph\n",
       "ODG_087 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph\n",
       "ODG_088 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph\n",
       "ODG_089 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph\n",
       "ODG_090 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph\n",
       "ODG_091 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph\n",
       "ODG_092 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph\n",
       "ODG_093 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph\n",
       "ODG_094 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph\n",
       "ODG_095 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph\n",
       "ODG_100 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph\n",
       "ODG_101 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph\n",
       "ODG_102 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph\n",
       "ODG_103 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph\n",
       "ODG_104 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph\n",
       "ODG_105 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph\n",
       "ODG_106 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph\n",
       "ODG_107 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph\n",
       "ODG_108 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph\n",
       "ODG_109 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning message in dir.create(outdir):\n",
      "“'out/ipynb/compute_simple_stats' already exists”\n"
     ]
    }
   ],
   "source": [
    "outdir <- \"out/ipynb/compute_simple_stats\"\n",
    "outfile <- file.path(\n",
    "    outdir,\n",
    "    paste0(\n",
    "        context,\n",
    "        \".rds\"\n",
    "    )\n",
    ")\n",
    "\n",
    "if (file.exists(outfile)) {\n",
    "    sample_metadata <-readRDS(outfile)\n",
    "} else {\n",
    "    # We create an output sample_metadata that will hold the results\n",
    "    sample_metadata$mean_methylation <- NA\n",
    "\n",
    "    dir.create(outdir)\n",
    "    saveRDS(\n",
    "        sample_metadata,\n",
    "        file = outfile\n",
    "    )\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style>\n",
       ".list-inline {list-style: none; margin:0; padding: 0}\n",
       ".list-inline>li {display: inline-block}\n",
       ".list-inline>li:not(:last-child)::after {content: \"\\00b7\"; padding: 0 .5ex}\n",
       "</style>\n",
       "<ol class=list-inline><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li><li>&lt;NA&gt;</li></ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\item <NA>\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. &lt;NA&gt;\n",
       "2. &lt;NA&gt;\n",
       "3. &lt;NA&gt;\n",
       "4. &lt;NA&gt;\n",
       "5. &lt;NA&gt;\n",
       "6. &lt;NA&gt;\n",
       "7. &lt;NA&gt;\n",
       "8. &lt;NA&gt;\n",
       "9. &lt;NA&gt;\n",
       "10. &lt;NA&gt;\n",
       "11. &lt;NA&gt;\n",
       "12. &lt;NA&gt;\n",
       "13. &lt;NA&gt;\n",
       "14. &lt;NA&gt;\n",
       "15. &lt;NA&gt;\n",
       "16. &lt;NA&gt;\n",
       "17. &lt;NA&gt;\n",
       "18. &lt;NA&gt;\n",
       "19. &lt;NA&gt;\n",
       "20. &lt;NA&gt;\n",
       "21. &lt;NA&gt;\n",
       "22. &lt;NA&gt;\n",
       "23. &lt;NA&gt;\n",
       "24. &lt;NA&gt;\n",
       "25. &lt;NA&gt;\n",
       "26. &lt;NA&gt;\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       " [1] NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA\n",
       "[26] NA"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sample_metadata$mean_methylation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading sample 1/26\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading sample 2/26\n",
      "\n",
      "Loading sample 3/26\n",
      "\n",
      "Loading sample 4/26\n",
      "\n",
      "Loading sample 5/26\n",
      "\n",
      "Loading sample 6/26\n",
      "\n",
      "Loading sample 7/26\n",
      "\n",
      "Loading sample 8/26\n",
      "\n",
      "Loading sample 9/26\n",
      "\n",
      "Loading sample 10/26\n",
      "\n",
      "Loading sample 11/26\n",
      "\n",
      "Loading sample 12/26\n",
      "\n",
      "Loading sample 13/26\n",
      "\n",
      "Loading sample 14/26\n",
      "\n",
      "Loading sample 15/26\n",
      "\n",
      "Loading sample 16/26\n",
      "\n",
      "Loading sample 17/26\n",
      "\n",
      "Loading sample 18/26\n",
      "\n",
      "Loading sample 19/26\n",
      "\n",
      "Loading sample 20/26\n",
      "\n",
      "Loading sample 21/26\n",
      "\n",
      "Loading sample 22/26\n",
      "\n",
      "Loading sample 23/26\n",
      "\n",
      "Loading sample 24/26\n",
      "\n",
      "Loading sample 25/26\n",
      "\n",
      "Loading sample 26/26\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 26 × 7</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>sample</th><th scope=col>fastq_1</th><th scope=col>fastq_2</th><th scope=col>patient_number</th><th scope=col>tissue</th><th scope=col>bedGraph</th><th scope=col>mean_methylation</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ODG_080</th><td>ODG_080</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz </td><td> 1</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph</td><td>76.18752</td></tr>\n",
       "\t<tr><th scope=row>ODG_081</th><td>ODG_081</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz </td><td> 1</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph</td><td>76.65098</td></tr>\n",
       "\t<tr><th scope=row>ODG_082</th><td>ODG_082</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz </td><td> 2</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph</td><td>75.34786</td></tr>\n",
       "\t<tr><th scope=row>ODG_083</th><td>ODG_083</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz </td><td> 2</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph</td><td>69.43403</td></tr>\n",
       "\t<tr><th scope=row>ODG_084</th><td>ODG_084</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz </td><td> 3</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph</td><td>77.39842</td></tr>\n",
       "\t<tr><th scope=row>ODG_085</th><td>ODG_085</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz </td><td> 3</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph</td><td>71.54112</td></tr>\n",
       "\t<tr><th scope=row>ODG_086</th><td>ODG_086</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz </td><td> 4</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph</td><td>76.13270</td></tr>\n",
       "\t<tr><th scope=row>ODG_087</th><td>ODG_087</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz </td><td> 4</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph</td><td>72.05458</td></tr>\n",
       "\t<tr><th scope=row>ODG_088</th><td>ODG_088</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz </td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz </td><td> 5</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph</td><td>75.70387</td></tr>\n",
       "\t<tr><th scope=row>ODG_089</th><td>ODG_089</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz</td><td> 5</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph</td><td>71.75048</td></tr>\n",
       "\t<tr><th scope=row>ODG_090</th><td>ODG_090</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz</td><td> 6</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph</td><td>76.92792</td></tr>\n",
       "\t<tr><th scope=row>ODG_091</th><td>ODG_091</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz</td><td> 6</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph</td><td>74.59039</td></tr>\n",
       "\t<tr><th scope=row>ODG_092</th><td>ODG_092</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz</td><td> 7</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph</td><td>77.68733</td></tr>\n",
       "\t<tr><th scope=row>ODG_093</th><td>ODG_093</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz</td><td> 7</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph</td><td>70.04719</td></tr>\n",
       "\t<tr><th scope=row>ODG_094</th><td>ODG_094</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz</td><td> 8</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph</td><td>75.09398</td></tr>\n",
       "\t<tr><th scope=row>ODG_095</th><td>ODG_095</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz</td><td> 8</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph</td><td>70.51608</td></tr>\n",
       "\t<tr><th scope=row>ODG_100</th><td>ODG_100</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz</td><td>11</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph</td><td>75.95879</td></tr>\n",
       "\t<tr><th scope=row>ODG_101</th><td>ODG_101</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz</td><td>11</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph</td><td>79.08287</td></tr>\n",
       "\t<tr><th scope=row>ODG_102</th><td>ODG_102</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz</td><td>12</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph</td><td>75.81587</td></tr>\n",
       "\t<tr><th scope=row>ODG_103</th><td>ODG_103</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz</td><td>12</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph</td><td>74.42226</td></tr>\n",
       "\t<tr><th scope=row>ODG_104</th><td>ODG_104</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz</td><td>13</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph</td><td>75.76062</td></tr>\n",
       "\t<tr><th scope=row>ODG_105</th><td>ODG_105</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz</td><td>13</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph</td><td>75.73460</td></tr>\n",
       "\t<tr><th scope=row>ODG_106</th><td>ODG_106</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz</td><td>14</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph</td><td>76.60429</td></tr>\n",
       "\t<tr><th scope=row>ODG_107</th><td>ODG_107</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz</td><td>14</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph</td><td>72.89219</td></tr>\n",
       "\t<tr><th scope=row>ODG_108</th><td>ODG_108</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz</td><td>15</td><td>normal </td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph</td><td>74.72216</td></tr>\n",
       "\t<tr><th scope=row>ODG_109</th><td>ODG_109</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz</td><td>inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz</td><td>15</td><td>tumoral</td><td>out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph</td><td>69.08381</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 26 × 7\n",
       "\\begin{tabular}{r|lllllll}\n",
       "  & sample & fastq\\_1 & fastq\\_2 & patient\\_number & tissue & bedGraph & mean\\_methylation\\\\\n",
       "  & <chr> & <chr> & <chr> & <int> & <chr> & <chr> & <dbl>\\\\\n",
       "\\hline\n",
       "\tODG\\_080 & ODG\\_080 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_080\\_S1\\_R2\\_001.fastq.gz  &  1 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_080.markdup.sorted\\_CpG.bedGraph & 76.18752\\\\\n",
       "\tODG\\_081 & ODG\\_081 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_081\\_S2\\_R2\\_001.fastq.gz  &  1 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_081.markdup.sorted\\_CpG.bedGraph & 76.65098\\\\\n",
       "\tODG\\_082 & ODG\\_082 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_082\\_S3\\_R2\\_001.fastq.gz  &  2 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_082.markdup.sorted\\_CpG.bedGraph & 75.34786\\\\\n",
       "\tODG\\_083 & ODG\\_083 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_083\\_S4\\_R2\\_001.fastq.gz  &  2 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_083.markdup.sorted\\_CpG.bedGraph & 69.43403\\\\\n",
       "\tODG\\_084 & ODG\\_084 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_084\\_S5\\_R2\\_001.fastq.gz  &  3 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_084.markdup.sorted\\_CpG.bedGraph & 77.39842\\\\\n",
       "\tODG\\_085 & ODG\\_085 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_085\\_S6\\_R2\\_001.fastq.gz  &  3 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_085.markdup.sorted\\_CpG.bedGraph & 71.54112\\\\\n",
       "\tODG\\_086 & ODG\\_086 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_086\\_S7\\_R2\\_001.fastq.gz  &  4 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_086.markdup.sorted\\_CpG.bedGraph & 76.13270\\\\\n",
       "\tODG\\_087 & ODG\\_087 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_087\\_S8\\_R2\\_001.fastq.gz  &  4 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_087.markdup.sorted\\_CpG.bedGraph & 72.05458\\\\\n",
       "\tODG\\_088 & ODG\\_088 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R1\\_001.fastq.gz  & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_088\\_S9\\_R2\\_001.fastq.gz  &  5 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_088.markdup.sorted\\_CpG.bedGraph & 75.70387\\\\\n",
       "\tODG\\_089 & ODG\\_089 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_089\\_S10\\_R2\\_001.fastq.gz &  5 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_089.markdup.sorted\\_CpG.bedGraph & 71.75048\\\\\n",
       "\tODG\\_090 & ODG\\_090 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_090\\_S11\\_R2\\_001.fastq.gz &  6 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_090.markdup.sorted\\_CpG.bedGraph & 76.92792\\\\\n",
       "\tODG\\_091 & ODG\\_091 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_091\\_S12\\_R2\\_001.fastq.gz &  6 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_091.markdup.sorted\\_CpG.bedGraph & 74.59039\\\\\n",
       "\tODG\\_092 & ODG\\_092 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_092\\_S13\\_R2\\_001.fastq.gz &  7 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_092.markdup.sorted\\_CpG.bedGraph & 77.68733\\\\\n",
       "\tODG\\_093 & ODG\\_093 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_093\\_S14\\_R2\\_001.fastq.gz &  7 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_093.markdup.sorted\\_CpG.bedGraph & 70.04719\\\\\n",
       "\tODG\\_094 & ODG\\_094 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_094\\_S15\\_R2\\_001.fastq.gz &  8 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_094.markdup.sorted\\_CpG.bedGraph & 75.09398\\\\\n",
       "\tODG\\_095 & ODG\\_095 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_095\\_S16\\_R2\\_001.fastq.gz &  8 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_095.markdup.sorted\\_CpG.bedGraph & 70.51608\\\\\n",
       "\tODG\\_100 & ODG\\_100 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_100\\_S21\\_R2\\_001.fastq.gz & 11 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_100.markdup.sorted\\_CpG.bedGraph & 75.95879\\\\\n",
       "\tODG\\_101 & ODG\\_101 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_101\\_S22\\_R2\\_001.fastq.gz & 11 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_101.markdup.sorted\\_CpG.bedGraph & 79.08287\\\\\n",
       "\tODG\\_102 & ODG\\_102 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_102\\_S23\\_R2\\_001.fastq.gz & 12 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_102.markdup.sorted\\_CpG.bedGraph & 75.81587\\\\\n",
       "\tODG\\_103 & ODG\\_103 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_103\\_S24\\_R2\\_001.fastq.gz & 12 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_103.markdup.sorted\\_CpG.bedGraph & 74.42226\\\\\n",
       "\tODG\\_104 & ODG\\_104 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_104\\_S25\\_R2\\_001.fastq.gz & 13 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_104.markdup.sorted\\_CpG.bedGraph & 75.76062\\\\\n",
       "\tODG\\_105 & ODG\\_105 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_105\\_S26\\_R2\\_001.fastq.gz & 13 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_105.markdup.sorted\\_CpG.bedGraph & 75.73460\\\\\n",
       "\tODG\\_106 & ODG\\_106 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_106\\_S27\\_R2\\_001.fastq.gz & 14 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_106.markdup.sorted\\_CpG.bedGraph & 76.60429\\\\\n",
       "\tODG\\_107 & ODG\\_107 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_107\\_S28\\_R2\\_001.fastq.gz & 14 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_107.markdup.sorted\\_CpG.bedGraph & 72.89219\\\\\n",
       "\tODG\\_108 & ODG\\_108 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_108\\_S29\\_R2\\_001.fastq.gz & 15 & normal  & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_108.markdup.sorted\\_CpG.bedGraph & 74.72216\\\\\n",
       "\tODG\\_109 & ODG\\_109 & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R1\\_001.fastq.gz & inp/ONCODIAG/Novaseq\\_131222/fastq/ODG\\_109\\_S30\\_R2\\_001.fastq.gz & 15 & tumoral & out/methyldackel/extract\\_fa-genome-hg19/samtools/index/nextflow/nfcore\\_methylseq/bwameth/deduplicated/ODG\\_109.markdup.sorted\\_CpG.bedGraph & 69.08381\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 26 × 7\n",
       "\n",
       "| <!--/--> | sample &lt;chr&gt; | fastq_1 &lt;chr&gt; | fastq_2 &lt;chr&gt; | patient_number &lt;int&gt; | tissue &lt;chr&gt; | bedGraph &lt;chr&gt; | mean_methylation &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|\n",
       "| ODG_080 | ODG_080 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz  |  1 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph | 76.18752 |\n",
       "| ODG_081 | ODG_081 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz  |  1 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph | 76.65098 |\n",
       "| ODG_082 | ODG_082 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz  |  2 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph | 75.34786 |\n",
       "| ODG_083 | ODG_083 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz  |  2 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph | 69.43403 |\n",
       "| ODG_084 | ODG_084 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz  |  3 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph | 77.39842 |\n",
       "| ODG_085 | ODG_085 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz  |  3 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph | 71.54112 |\n",
       "| ODG_086 | ODG_086 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz  |  4 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph | 76.13270 |\n",
       "| ODG_087 | ODG_087 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz  |  4 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph | 72.05458 |\n",
       "| ODG_088 | ODG_088 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz  | inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz  |  5 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph | 75.70387 |\n",
       "| ODG_089 | ODG_089 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz |  5 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph | 71.75048 |\n",
       "| ODG_090 | ODG_090 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz |  6 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph | 76.92792 |\n",
       "| ODG_091 | ODG_091 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz |  6 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph | 74.59039 |\n",
       "| ODG_092 | ODG_092 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz |  7 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph | 77.68733 |\n",
       "| ODG_093 | ODG_093 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz |  7 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph | 70.04719 |\n",
       "| ODG_094 | ODG_094 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz |  8 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph | 75.09398 |\n",
       "| ODG_095 | ODG_095 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz |  8 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph | 70.51608 |\n",
       "| ODG_100 | ODG_100 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz | 11 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph | 75.95879 |\n",
       "| ODG_101 | ODG_101 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz | 11 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph | 79.08287 |\n",
       "| ODG_102 | ODG_102 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz | 12 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph | 75.81587 |\n",
       "| ODG_103 | ODG_103 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz | 12 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph | 74.42226 |\n",
       "| ODG_104 | ODG_104 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz | 13 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph | 75.76062 |\n",
       "| ODG_105 | ODG_105 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz | 13 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph | 75.73460 |\n",
       "| ODG_106 | ODG_106 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz | 14 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph | 76.60429 |\n",
       "| ODG_107 | ODG_107 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz | 14 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph | 72.89219 |\n",
       "| ODG_108 | ODG_108 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz | 15 | normal  | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph | 74.72216 |\n",
       "| ODG_109 | ODG_109 | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz | inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz | 15 | tumoral | out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph | 69.08381 |\n",
       "\n"
      ],
      "text/plain": [
       "        sample  fastq_1                                                      \n",
       "ODG_080 ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R1_001.fastq.gz \n",
       "ODG_081 ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R1_001.fastq.gz \n",
       "ODG_082 ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R1_001.fastq.gz \n",
       "ODG_083 ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R1_001.fastq.gz \n",
       "ODG_084 ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R1_001.fastq.gz \n",
       "ODG_085 ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R1_001.fastq.gz \n",
       "ODG_086 ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R1_001.fastq.gz \n",
       "ODG_087 ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R1_001.fastq.gz \n",
       "ODG_088 ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R1_001.fastq.gz \n",
       "ODG_089 ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R1_001.fastq.gz\n",
       "ODG_090 ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R1_001.fastq.gz\n",
       "ODG_091 ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R1_001.fastq.gz\n",
       "ODG_092 ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R1_001.fastq.gz\n",
       "ODG_093 ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R1_001.fastq.gz\n",
       "ODG_094 ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R1_001.fastq.gz\n",
       "ODG_095 ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R1_001.fastq.gz\n",
       "ODG_100 ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R1_001.fastq.gz\n",
       "ODG_101 ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R1_001.fastq.gz\n",
       "ODG_102 ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R1_001.fastq.gz\n",
       "ODG_103 ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R1_001.fastq.gz\n",
       "ODG_104 ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R1_001.fastq.gz\n",
       "ODG_105 ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R1_001.fastq.gz\n",
       "ODG_106 ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R1_001.fastq.gz\n",
       "ODG_107 ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R1_001.fastq.gz\n",
       "ODG_108 ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R1_001.fastq.gz\n",
       "ODG_109 ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R1_001.fastq.gz\n",
       "        fastq_2                                                      \n",
       "ODG_080 inp/ONCODIAG/Novaseq_131222/fastq/ODG_080_S1_R2_001.fastq.gz \n",
       "ODG_081 inp/ONCODIAG/Novaseq_131222/fastq/ODG_081_S2_R2_001.fastq.gz \n",
       "ODG_082 inp/ONCODIAG/Novaseq_131222/fastq/ODG_082_S3_R2_001.fastq.gz \n",
       "ODG_083 inp/ONCODIAG/Novaseq_131222/fastq/ODG_083_S4_R2_001.fastq.gz \n",
       "ODG_084 inp/ONCODIAG/Novaseq_131222/fastq/ODG_084_S5_R2_001.fastq.gz \n",
       "ODG_085 inp/ONCODIAG/Novaseq_131222/fastq/ODG_085_S6_R2_001.fastq.gz \n",
       "ODG_086 inp/ONCODIAG/Novaseq_131222/fastq/ODG_086_S7_R2_001.fastq.gz \n",
       "ODG_087 inp/ONCODIAG/Novaseq_131222/fastq/ODG_087_S8_R2_001.fastq.gz \n",
       "ODG_088 inp/ONCODIAG/Novaseq_131222/fastq/ODG_088_S9_R2_001.fastq.gz \n",
       "ODG_089 inp/ONCODIAG/Novaseq_131222/fastq/ODG_089_S10_R2_001.fastq.gz\n",
       "ODG_090 inp/ONCODIAG/Novaseq_131222/fastq/ODG_090_S11_R2_001.fastq.gz\n",
       "ODG_091 inp/ONCODIAG/Novaseq_131222/fastq/ODG_091_S12_R2_001.fastq.gz\n",
       "ODG_092 inp/ONCODIAG/Novaseq_131222/fastq/ODG_092_S13_R2_001.fastq.gz\n",
       "ODG_093 inp/ONCODIAG/Novaseq_131222/fastq/ODG_093_S14_R2_001.fastq.gz\n",
       "ODG_094 inp/ONCODIAG/Novaseq_131222/fastq/ODG_094_S15_R2_001.fastq.gz\n",
       "ODG_095 inp/ONCODIAG/Novaseq_131222/fastq/ODG_095_S16_R2_001.fastq.gz\n",
       "ODG_100 inp/ONCODIAG/Novaseq_131222/fastq/ODG_100_S21_R2_001.fastq.gz\n",
       "ODG_101 inp/ONCODIAG/Novaseq_131222/fastq/ODG_101_S22_R2_001.fastq.gz\n",
       "ODG_102 inp/ONCODIAG/Novaseq_131222/fastq/ODG_102_S23_R2_001.fastq.gz\n",
       "ODG_103 inp/ONCODIAG/Novaseq_131222/fastq/ODG_103_S24_R2_001.fastq.gz\n",
       "ODG_104 inp/ONCODIAG/Novaseq_131222/fastq/ODG_104_S25_R2_001.fastq.gz\n",
       "ODG_105 inp/ONCODIAG/Novaseq_131222/fastq/ODG_105_S26_R2_001.fastq.gz\n",
       "ODG_106 inp/ONCODIAG/Novaseq_131222/fastq/ODG_106_S27_R2_001.fastq.gz\n",
       "ODG_107 inp/ONCODIAG/Novaseq_131222/fastq/ODG_107_S28_R2_001.fastq.gz\n",
       "ODG_108 inp/ONCODIAG/Novaseq_131222/fastq/ODG_108_S29_R2_001.fastq.gz\n",
       "ODG_109 inp/ONCODIAG/Novaseq_131222/fastq/ODG_109_S30_R2_001.fastq.gz\n",
       "        patient_number tissue \n",
       "ODG_080  1             normal \n",
       "ODG_081  1             tumoral\n",
       "ODG_082  2             normal \n",
       "ODG_083  2             tumoral\n",
       "ODG_084  3             normal \n",
       "ODG_085  3             tumoral\n",
       "ODG_086  4             normal \n",
       "ODG_087  4             tumoral\n",
       "ODG_088  5             normal \n",
       "ODG_089  5             tumoral\n",
       "ODG_090  6             normal \n",
       "ODG_091  6             tumoral\n",
       "ODG_092  7             normal \n",
       "ODG_093  7             tumoral\n",
       "ODG_094  8             normal \n",
       "ODG_095  8             tumoral\n",
       "ODG_100 11             normal \n",
       "ODG_101 11             tumoral\n",
       "ODG_102 12             normal \n",
       "ODG_103 12             tumoral\n",
       "ODG_104 13             normal \n",
       "ODG_105 13             tumoral\n",
       "ODG_106 14             normal \n",
       "ODG_107 14             tumoral\n",
       "ODG_108 15             normal \n",
       "ODG_109 15             tumoral\n",
       "        bedGraph                                                                                                                                 \n",
       "ODG_080 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_080.markdup.sorted_CpG.bedGraph\n",
       "ODG_081 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_081.markdup.sorted_CpG.bedGraph\n",
       "ODG_082 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_082.markdup.sorted_CpG.bedGraph\n",
       "ODG_083 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_083.markdup.sorted_CpG.bedGraph\n",
       "ODG_084 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_084.markdup.sorted_CpG.bedGraph\n",
       "ODG_085 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_085.markdup.sorted_CpG.bedGraph\n",
       "ODG_086 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_086.markdup.sorted_CpG.bedGraph\n",
       "ODG_087 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_087.markdup.sorted_CpG.bedGraph\n",
       "ODG_088 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_088.markdup.sorted_CpG.bedGraph\n",
       "ODG_089 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_089.markdup.sorted_CpG.bedGraph\n",
       "ODG_090 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_090.markdup.sorted_CpG.bedGraph\n",
       "ODG_091 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_091.markdup.sorted_CpG.bedGraph\n",
       "ODG_092 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_092.markdup.sorted_CpG.bedGraph\n",
       "ODG_093 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_093.markdup.sorted_CpG.bedGraph\n",
       "ODG_094 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_094.markdup.sorted_CpG.bedGraph\n",
       "ODG_095 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_095.markdup.sorted_CpG.bedGraph\n",
       "ODG_100 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_100.markdup.sorted_CpG.bedGraph\n",
       "ODG_101 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_101.markdup.sorted_CpG.bedGraph\n",
       "ODG_102 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_102.markdup.sorted_CpG.bedGraph\n",
       "ODG_103 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_103.markdup.sorted_CpG.bedGraph\n",
       "ODG_104 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_104.markdup.sorted_CpG.bedGraph\n",
       "ODG_105 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_105.markdup.sorted_CpG.bedGraph\n",
       "ODG_106 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_106.markdup.sorted_CpG.bedGraph\n",
       "ODG_107 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_107.markdup.sorted_CpG.bedGraph\n",
       "ODG_108 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_108.markdup.sorted_CpG.bedGraph\n",
       "ODG_109 out/methyldackel/extract_fa-genome-hg19/samtools/index/nextflow/nfcore_methylseq/bwameth/deduplicated/ODG_109.markdup.sorted_CpG.bedGraph\n",
       "        mean_methylation\n",
       "ODG_080 76.18752        \n",
       "ODG_081 76.65098        \n",
       "ODG_082 75.34786        \n",
       "ODG_083 69.43403        \n",
       "ODG_084 77.39842        \n",
       "ODG_085 71.54112        \n",
       "ODG_086 76.13270        \n",
       "ODG_087 72.05458        \n",
       "ODG_088 75.70387        \n",
       "ODG_089 71.75048        \n",
       "ODG_090 76.92792        \n",
       "ODG_091 74.59039        \n",
       "ODG_092 77.68733        \n",
       "ODG_093 70.04719        \n",
       "ODG_094 75.09398        \n",
       "ODG_095 70.51608        \n",
       "ODG_100 75.95879        \n",
       "ODG_101 79.08287        \n",
       "ODG_102 75.81587        \n",
       "ODG_103 74.42226        \n",
       "ODG_104 75.76062        \n",
       "ODG_105 75.73460        \n",
       "ODG_106 76.60429        \n",
       "ODG_107 72.89219        \n",
       "ODG_108 74.72216        \n",
       "ODG_109 69.08381        "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for (row in 1:nrow(sample_metadata)) {\n",
    "    message(\"Loading sample \", row, \"/\", nrow(sample_metadata))\n",
    "    if (is.na(sample_metadata[row,]$mean_methylation)) {\n",
    "        d <- fread(\n",
    "            sample_metadata[row,]$bedGraph,\n",
    "            select = 4,\n",
    "            skip = 1,\n",
    "            sep = \"\\t\"\n",
    "        )\n",
    "        sample_metadata[row,]$mean_methylation <- mean(d$V4)\n",
    "        rm(d)\n",
    "        saveRDS(\n",
    "            sample_metadata,\n",
    "            file = outfile\n",
    "        )\n",
    "    }\n",
    "\n",
    "}\n",
    "sample_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlgAAAJYCAMAAACJuGjuAAADAFBMVEUAAAAAAP8BAQECAgIDAwMEBAQFBQUGBgYHBwcICAgJCQkKCgoLCwsMDAwNDQ0ODg4PDw8QEBARERESEhITExMUFBQVFRUWFhYXFxcYGBgZGRkaGhobGxscHBwdHR0eHh4fHx8gICAhISEiIiIjIyMkJCQlJSUmJiYnJycoKCgpKSkqKiorKyssLCwtLS0uLi4vLy8wMDAxMTEyMjIzMzM0NDQ1NTU2NjY3Nzc4ODg5OTk6Ojo7Ozs8PDw9PT0+Pj4/Pz9AQEBBQUFCQkJDQ0NERERFRUVGRkZHR0dISEhJSUlKSkpLS0tMTExNTU1OTk5PT09QUFBRUVFSUlJTU1NUVFRVVVVWVlZXV1dYWFhZWVlaWlpbW1tcXFxdXV1eXl5fX19gYGBhYWFiYmJjY2NkZGRlZWVmZmZnZ2doaGhpaWlqampra2tsbGxubm5vb29wcHBxcXFycnJzc3N0dHR1dXV2dnZ3d3d4eHh5eXl6enp7e3t8fHx9fX1+fn5/f3+AgICBgYGCgoKDg4OEhISFhYWGhoaHh4eIiIiJiYmKioqLi4uMjIyNjY2Ojo6Pj4+QkJCRkZGSkpKTk5OUlJSVlZWWlpaXl5eYmJiZmZmampqbm5udnZ2enp6fn5+goKChoaGioqKjo6OkpKSlpaWmpqanp6eoqKipqamqqqqrq6usrKytra2urq6vr6+wsLCxsbGysrKzs7O0tLS1tbW2tra3t7e4uLi5ubm6urq7u7u8vLy9vb2+vr6/v7/AwMDBwcHCwsLDw8PExMTFxcXGxsbHx8fIyMjJycnKysrLy8vMzMzNzc3Ozs7Pz8/Q0NDR0dHS0tLT09PU1NTV1dXW1tbX19fY2NjZ2dna2trb29vc3Nzd3d3e3t7f39/g4ODh4eHi4uLj4+Pk5OTl5eXm5ubn5+fo6Ojp6enq6urr6+vs7Ozt7e3u7u7v7+/w8PDx8fHy8vLz8/P09PT19fX29vb39/f4+Pj5+fn6+vr7+/v8/Pz9/f3+/v7/AAD///9u/oFqAAAACXBIWXMAABJ0AAASdAHeZh94AAAgAElEQVR4nO2dCXxU5b3+X0IEwVatG9qLuLbaSm+tS69traJVrxYNqxEXXJFSlXIt3qJXbMRWEfelF8GFesHlVm4RVAT5K8UdEUEEAwiKiA0YlkiAhCzD+z/nZMLMmSdnfmcyM8xz4u/7+ZhM8h0e8HmfDzMZMhNjFSUPmEL/AZS2iQ5LyQs6LCUv6LCUvKDDUvKCDkvJCzosJS/osJS8kNNhxTZuTq+r0+nG9Lph45as9NZ0ul7S29LpOknXpNPbJV2bTtem1wUjt8Oq/DqdbpR02lk2SDrtLOvT67rKtLOsq0y7u+2STru7Wkmn3V1Nel0wdFgeOqxco8Py0GHlGh2Whw4r1+iwPHRYuUaH5aHDyjU6LA8dVq7RYXnosHKNDstDh5VrdFgeOqxco8Py0GHlGh2Whw4r18jDWv/S8xXNl9dOf2FVmqvqsAK0DgtZWjpyZP/5TZfnl466ve/LwdfVYQVoHRYybLy1E66OeZeHTrJ26oBY4HV1WAFahwVUlKxybg1LlnsfDPi7tf/oWx94ZR1WgNZhAXN7NThvS1/1Pni49PH/uWxa8JV1WAFahwW8Vuq+vaJpTVVDb/7jpW/udFWVCiXBd1Z2HdKwXvWGdfmL7tv6616w9vXeXzS76k2pVG6ATyWxsXJ9Ol3IX71R+tWF1BvT6Q0t/OooDKvppvCC17zLfdzLV04JvLLeFAZovSkEKkqcv6A2lKx0L7/e2/2+/UEvBF5ZhxWgdVjI0HHWPjm40cYa7bo+M62d33tV4HV1WAFah4Us7n/TLf0WWjv6emtf6Dfqjv6Tg6+rwwrQOqwWWDd12lrn3RsznDdfTH/xszRX1WEFaB1Wdnxzh7XDB2gdVnbosHRYzeiwPHRYuUaH5aHDyjU6LA8dVq7RYXnosHKNDstDh5VrdFgeOqxc8w0eVjsfOqzcosPSYeWFAg/Lf7g6rBa1DkuHpcNqRoelw8oLbXlY/nAd1i5Fh6XDygs5HlbagnVYLdLCsPx/NB2WDkuH1UzWw/J3oMPSYTXBPayMdqfDYkKH1cphZfD/rcNqDYUcVkbL2bXD8mthOVCaDkuHpcNqGR2WDisv5HtYfq3Dkn81ah1WmxqWX+uwMkSHFUeHlVt0WHGiNCyf1WG1K+ywhD+aDiu36LDi5HVYmWgM12HpsHRYzeiw4uiwcosOK44OK7fosOLosHKLDiuODiu36LDi6LByiw4rjg4rt+iw4uiwcosOK44OK7fosOIQDyu91mG102HpsMKTUYM6LB1WWDJqUIelwwpLRg3qsHRYYcmoweyGJYTrsJjQYcXRYeWWNjSs7LQOK7dQDSuvy9Fh7VJ0WHF0WLlFhxVHh5VbdFhxdFi5RYcVR4eVW3RYcXRYuUWHFUcY1q77owm/N2gdVjsdlg4rPBk1qMPSYYWgepNDugZBr9+UQka/+puhhVpAV25ILXVTLGfzaD3ZDCvmkq5B0F/HUsjoV38ztFAL6MptqaUy7EpvCum0UAvoNnhT6JFRgzosWQu1gNZhqQ6lhVJB67BUh9JCqaB1WKpDaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkcyAOq+HRgQMejv/Y4dhzV5aO/NLvSY8nulooFTRkcyAO6/Grl3w8+MGmy3+7cO7q266J+Tzp8URXC6WChmwOpGHV9H/b2oW9q9zLdQNetbZy8BrfFUiPJ7paKBU0ZHMgDWtJyWbn5rDXXPfy4j41eAXS44muFkoFDdkcSMN6va/7duB09+2sK98beuGtFTtd7TaHdA2qboUWSgUN2du2xQIOc1ciDWvWAPftlVPdt1NLf/v2RzcNqm12VZUO6RpU3QotlAoasisrozCs1/u5bwfOcN++WLLY2s393mx2dbUO6RpU3QotlAoasmtrgw5zVyLfx6p27mP1nudefrtki/P26im+K6RrUHUrtFAqaMjmQBpWbX/nfvuiPt5XhZt6LbR2S793fFcgPZ7oaqFU0JDNgfg41virFpUPeci5TXTuv/9l8KLPRw2p83nS44muFkoFDdkciMOqGzvgor9st3b09c5N4oSB549a5/ekxxNdLZQKGrI50H8rZNNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDnRYbFooFTRkc5AYVlVlQ9Prx3jUpfk1PkiPJ7paKBU0ZHOQGFYPU+7818zssAGkxxNdLZQKGrI5SAxr/IivnP+aWRk2gPR4oquFUkFDNgdwH6vBuVHcmkEA6fFEVwulgoZsDlKGNefYJ6x9Yo8RDaEDSI8nulooFTRkc+Af1tKOB0+39oMBZnjoANLjia4WSgUN2Rz4h/W7A9d778s6bA4bQHo80dVCqaAhmwP/sH59YdP7Jeb9sAGkxxNdLZQKGrI58A/rohOb3s8wS8IGkB5PdLVQKmjI5sA/rElmjHu3fW33Q0O/ojPp8URXC6WChmwO/MNqONMc0vOSX3XqMCN0AOnxRFcLpYKGbA5SHm6ou+u4zma/fvPDB5AeT3S1UCpoyOaghX+EDv8Ylgvp8URXC6WChmwOUoe18Pqzzxr4RAs/jCkI0uOJrhZKBQ3ZHKQM6/72puM+xnRfGzqA9Hiiq4VSQUM2B/5hvVt08vuN9usHOvUJHUB6PNHVQqmgIZsD/7CGHOj9bBN7T/uvwgaQHk90tVAqaMjmwD+ss/o2vV9s3g0bQHo80dVCqaAhmwP/sM4/vun9S/rIuw4rO/zDetzc7777Z/eDG8MGkB5PdLVQKmjI5sA/rPpTzI8H/b7/HkV/Dx1AejzR1UKpoCGbg5SHG7YOP8CY9j9/JXwA6fFEVwulgoZsDuCR99i6VZl8ZzLr8URXC6WChmwOEsPaXpOM3scqlBZKBQ3ZHCQ//SuZ2WEDSI8nulooFTRkc5AY1gPXJbM8bADp8URXC6WChmwO9Cn2bFooFTRkc6DDYtNCqaAhmwMdFpsWSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHiWEN6Wvt2x9n8mvr6xzSNai6FVooFTRk14V+Cap8khjW2d2t7RH+qfUO26od0jWouhVaKBU0ZFdXh37uXh5JDOs35pr7jjz9/jhfhA1I16DqVmihVNCQzUFiWJ/9QB95Z9BCqaAhm4OkO++x1Ut/emV5nND/EE16PNHVQqmgIZsD/1eFD0zJOID0eKKrhVJBQzYH+rxCNi2UChqyOdDnFbJpoVTQkM2BPq+QTQulgoZsDvR5hWxaKBU0ZHOgzytk00KpoCGbA31eIZsWSgUN2Rzo8wrZtFAqaMjmQJ9XyKaFUkFDNgf6vEI2LZQKGrI50OcVsmmhVNCQzYF+ox+bFkoFDdkc6LDYtFAqaMjmQIfFpoVSQUM2BzosNi2UChqyOdBhsWmhVNCQzUHqsDYvWbjApTpsAOnxRFcLpYKGbA5SHiC9ppN+a3KBtVAqaMjmwD+ssabnmIc91oQNID2e6GqhVNCQzYF/WL3OyziA9Hiiq4VSQUM2B/5h9bwl4wDS44muFkoFDdkh+KQs9HevtBb/sG7tkXEA6fFEVwulgoZsgTfutfZl81zGB50h/mFtPPLahes3udSHDSA9nuhqoVTQkC1QNMja1Y+sDHu+rcU/rL576RNWC62FUkFDtnTig8IebVakfFU4vJkVYQNIjye6WigVNGSnZV2ZOa7sC+8+1vYX7rlvjvsyD80XlpR94nz0edkC523VU3c8GvqRgZbQR97ZtFAqaMhOy4oTzP4nfOTex6o6zhy0jzl1W+LCc+Zl5xpvmAnWztrPHNi+06NZ7EKfsMqmhVJBQ7Z04oOa7ryPKH7Hxu4x4xIXEsNavefxK2zV+e3fDL8D+G38H+oTVguuhVJBQ7Z04vFhXdbxU2vrJ32SuJAY1s1msXNpy7dLQ88AfxvfR/qE1cJroVTQkC2deHxY04v26PfYP51P7LyQGNbpxd5rsu9/eOgZ4G/j+0ifsFp4LZQKGrKlE48Py758VgdTdM7qxIXEsH7S+WSPnmFX0MJv4/tIn7BaeC2UChqypRMftPMB0s1TB5ozExcmm5es6ybYU74V+ul/gb+N7yN9wmrhtVAqaMiWTjw+rN9d7n70iwMSF140f3Uu/NkZ1u/NDOfSlmN/H3YFLfw2vo/0CauF10KpoCFboPMpX9W5wxpW9Fdr5+15TuJCxW7Hr7XT93WGtbzToe/b6gvNpLArQPQJq2xaKBU0ZAucbswUd1ibTzJ7H2S+vzJxwY4wRZ07PuQ+jvX83uaA3YtvDL0CRJ+wyqaFUkFDtkDVuDFLvEfeY68+dOe07Tbpgn3v7gdX1niPvG+cdMdjy8KvANEnrLJpoVTQkM1BYlhVlQ3Of82EfhF60uOJrhZKBQ3ZHCT/IMzypB+GOTtsAOnxRFcLpYKGbA4Swxo/4ivnv2ZCf78O6fFEVwulgoZsDgK+u+FLvSkslBZKBQ3ZHKQ88h7/AnOyWRA2gPR4oquFUkFDNgdJP8V+0sSJP+g50eV/zjXvhw0gPZ7oaqFU0JDNQdLfWAOTfpTOcQ1hA0iPJ7paKBU0ZHOQNKxNs2cfe8FslzkLwv/IO9Ljia4WSgUN2Rz472P9PfNvGSQ9nuhqoVTQkM1B6leFG5eXl5cveX98edgA0uOJrhZKBQ3ZHPiHtf3iYn2AtMBaKBU0ZGd0YMmEPfJQ+Ic1zpxxW8dLbzrOPKJ33gulhVJBQ3ZGB5ZM2CMPhX9YpUfU25NvtA29fho6gPR4oquFUkFDdkYHlkzoMw+Df1hnXmDtb86ydp7R73kvlBZKBQ3ZGR1YMmGPPBT+YfXvae1dXaytNPPCBpAeT3S1UCpoyM7owJIJe+Sh8A/r7k7z7HQz175o9KvCQmmhVNCQndGBJRP2yEOR8mozXYueq+u2b589D0+82sz6l56vSLrG0yl360mPJ7paKBU0ZAu0CySz5QikPI61+tp37YdHmiMTz/5aWjpyZP/5Oz+8rWR7uj8nzfFEVwulgoZsgXaBZLYcgZa+bSb2ddIHw8ZbO+HqWPyjmaU6rDxroVTQkC3QLpDQo0mexxkBIum7G2qSaX76V0XJKufWsGR500frLp6mw8qzFkoFDdkC7QKRfy0SYlg9TDKz45+d28u9T1X6qvdB7Ma/LdJh5VkLpYKGbIF2gSRdaWV548IPvL9cNsz93Hm7YunmBTXOJ+cvitm1c71btDVvLWoINawHrksm/leUfc17wZErpnkf/H14Y/Kwvl7vkK5B1a3QQqmgIXv9+phNQ7tAkq404vTuJ+579Fpbd03xv7Tvu8UOP2P/4rEjzjjhZ3ufccfhR3/nPVt1dqfDOh6xOsywXNaBf9Ub1uUvum9XX7jGJg+r2n2x0nQNqm6FFkoFDdmbNuVgWGaWrTp4lL1lnw/s8m432OFm+ratI4resu+Ys+vtORfaP3x/vf2q23+FHdZpx9/7pd833RRe8Jp7eeTvJk68t+TJj9P8Of3/l6pboYVSQUO2QLtAkod1lHvql9jDbnbej93HDu/mfvKH1taap6y95Rf202XWVp16ZdhhPXmCaX/GE8lfFFaUfOHczpZ4T9oZP2rUqBtKbn1bh9X2h3Wq8+aSAbb90877OWbD8JPdT55ibb37UkdlJ9nae8885Nv7Xh52WNYuu/Vo06n0+cTt3dBxzt4GN9pY09eJeuc935pjWD2ahnWgc/p2mtk2vEf8k83DGtTtycV1vTIYlsP8m48r2mfIO/GPFve/6ZZ+C60dfb0Oa5doqmGdd47z/sofWRhWlzut3frdSzMaVsW483Yv+o45aXHTh+umTnNfkfSNGU0f6T/p5FlTDWtR59InBnWcjsPq+b2nJp60R9/ww9rw6K/am2NuX9Uw7aADt8h/Strjia6mGNb4Yc6bP49ybrB+c9bgD5z778Pin2zoMdfaCdfYDX845+IJU/rbh4Nem80/rKk9O5gD/6PpOYX3mfd0WAXQFMPKHv+wenzrkpebb+qe7bcpTADp8URXC6WChuyMDiyZMMcdGv+wXg916+eD9Hiiq4VSQUN2RgeWTMZnnw78mdDvz3PZHDaA9Hiiq4VSQUN2RgeWTNgjD4V/WA1X76ZP/yqwFkoFDdkZHVgyYY88FP5hPWJOvP0Rjy8Dri/8OWmOJ7paKBU0ZGd0YMmEPfJQ+IfV59DaTANIjye6WigVNGRndGDJZHr0afEP69zMf8YF6fFEVwulgoZsDvzDuufAbZkGkB5PdLVQKmjI5iAxLPe7qyp+fOZb6/RnQuuwskf61mQR0uOJrhZKBQ3ZAjsCCXvkoUgMK/HzoPVnQuuwskZfNZlNC6WChmyBHYGEPfJQ6Ksms2mhVNCQLbAjkLBHHgp91WQ2LZQKGrIFdgQS9shDoa+azKaFUkFDtsCOQMIeeSj0VZPZtFAqaMgW2BFI6DMPg75qMpsWSgUN2QI7Asn47NOR+lXhij9d8rR9KNS3+DVBejzR1UKpoCFbYEcgSVcqPzz8Alr+vveUYT2ymzE3rzOHhP/hmqTHE10tlAoasgV2BJJ0pReKQw8g1LDmFvVbcszNjU/vfWroVNLjia4WSgUN2QI7Aklc5/OLiso+eflJ59IzU+2sKZ/e+9A/GyePmeN8XP/8mCerrZ059c37F9uvJ93+0LJQwxpyWK3tfrO1E41+P1ahNMOwVp1fNGKZt5feg+yI7j+58cSuPS8aUvyUrfjXbhd/v9sqO/yYrj+fsrTLv11xXId3wwzrrAHWG9ZS804L19Vh7QrNMCzvpnDnsHavsCvNUGsHldjLjv7a1vy8jx3eYaW1Q0+LWfuToWGGNeCkpmH9rwl9J4v0eKKr+YZ1vLV15v+s/ePP7P73Op97tnNsuPuiIbax8dOp3a8IM6xJ5lF3WB8e/L2wu2I9nuhqvmElPQG6+Fnnc2+YyuGnOu9Xl3Tc/9+PDnj9Bv+wGs813ff88S+LO7yiwyqUphnW9e7T6nv4h3XYfc7nni2u855y/6uTP7f27FDDsvV3/cCYzr+eG3pXrMcTXU0xrBlFDXb0Qdvsis7+Yf32mC12+ylnNb2WQ5fbrF3xrYGhhuWwvaqxhesFQno80dVCqaAhW2BHIElXWl7c/ZXP9jqqb9fT/MOqPOrwy3546PKmYQ3bY/Cg/X94dqhhNc6b/IwHvmikDmvXaIph2dfuWGTXPHj/5zOn2lkTnF2UfWLt7MesrXn69qe2WDtzgnul6Xc++smC0dZ7wCv9sL76sX5rcqE1x7Cyxj+sGzrfM32Wx8awAaTHE10tlAoasgV2BBL2yEPhH9avr884gPR4oquFUkFDtsCOQDI++3T4h3XV0IwDSI8nulooFTRkC+wIJOOzT4d/WO/v827A9QIhPZ7oaqFU0JAtsCOQTI8+LSlfFV5bdMxZZ7t8GDaA9Hiiq4VSQUO2wI5Awh55KPzDmmKK9u/q8XbA9QHS44muFkoFDdkCOwIJe+Sh8A/r/O5fZBpAejzR1UKpoCGbA/+wzrkl4wDS44muFkoFDdkc+If1X+dmHEB6PNHVQqmgIZsD/7AqDx3y3lp9tRkdVvb4h9V3T/0nnUJroVTQkM2Bf1hj9dVmCq6FUkFDNgcBrzYTHtLjia4WSgUN2RzosNi0UCpoyOZAh8WmhVJBQzYHOiw2LZQKGrI50GGxaaFU0JDNgQ6LTQulgoZsDvCHNC1c4FIdNoD0eKKrhVJBQzYH/mHVX9NJHyAtsBZKBQ3ZHKQ8QGp6jnnYY03YANLjia4WSgUN2Rz4h9XrvIwDSI8nulooFTRkc+AfVk/9tpmCa6FU0JDNgX9Yt/bIOID0eKKrhVJBQzYH/mFtPPLahetDf9vMliqHdA2qboUWSgUN2VVVsfxsJSNSvm1mr0y+baaxwSFdg6pboYVSQUN2Q+jX6M8n+m0zbFooFTRkc6CPvLNpoVTQkM1B6rA2Li8vL1/y/vjysAGkxxNdLZQKGrI58A9r+8XF+sh7gbVQKmjI5sA/rHHmjNs6XnrTceYR/SFNhdJCqaAhmwP/sEqPqLcn32gbev00dADp8URXC6WChmwO/MM68wJrf3OWtfPMV2EDSI8nulooFTRkc+AfVv+e1t7VxdpKMy9sAOnxRFcLpYKGbA78w7q70zw73cy1Lxr9qrBQWigVNGRzkPJPOl2Lnqvrtm+fPQ/XZ0IXSgulgoZsDlIex1p97bv2wyPNkeFff430eKKrhVJBQzYHLT3yHvs6gwDS44muFkoFDdkc6E9YZdNCqaAhmwP9CatsWigVNGRzoD9hlU0LpYKGbA70J6yyaaFU0JDNgf6EVTYtlAoasjnQn7DKpoVSQUM2B/oTVtm0UCpoyOZAf8IqmxZKBQ3ZHOhPWGXTQqmgIZsD/QmrbFooFTRkc5AYVsVnydSEDSA9nuhqoVTQkM1BYlg9TDKzwwaQHk90tVAqaMjmIHlYnfsM1+cVFlwLpYKGbA4Sw5pz9QHmJ3/6ONMA0uOJrhZKBQ3ZHCTfeW94bWhXc8wtCzMKID2e6GqhVNCQzUHKV4Wxd244wnxvxHvhA0iPJ7paKBU0ZHPQwjf6LRj5I3PI4rABpMcTXS2UChqyOWhhWKse+FWxeT1sAOnxRFcLpYKGbA5Sh/XhqOOMOeHP4e/Dkx5PdLVQKmjI5iB5WI1v/P4IU3zq/Z9nEkB6PNHVQqmgIZuDxLBmD+piOv76sdBPgY5DejzR1UKpoCGbg+QHSPe97G8vN7MhbADp8URXC6WChmwO9J902LRQKmjI5iAxrBcfS+afYQNIjye6WigVNGRzoC8VyaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BOKyGRwcOeDj+Mmyb7r30otsq/J70eKKrhVJBQzYH4rAev3rJx4MfbLr8hxuWrxo1uNbnSY8nulooFTRkcyANq6b/29Yu7F3lXv6i5HNrq3st8F2B9Hiiq4VSQUM2B9KwlpRsdm4Oe3kvo7xtcczaLb3e9l2B9Hiiq4VSQUM2B9KwXu/rvh04fecnJl1Q3Xxxe41DugZVt0ILpYKG7JqaWH62khHSsGYNcN9eObX54zd7z9rpqiod0jWouhVaKBU0ZFdWRmFYr/dz3w6cEf9wVp/JCVdX65CuQdWt0EKpoCG7traFc9zlyPexnFu+ht7zmj56rvf01Cuka1B1K7RQKmjI5kAaVm1/5377oj7eV4X2+T5vwRVIjye6WigVNGRzID6ONf6qReVDHnJuE6fbNX3ume9Q6fOkxxNdLZQKGrI5EIdVN3bARX/Zbu3o6+3fSjxm+jzp8URXC6WChmwO9N8K2bRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LLgiK5sAAAihSURBVDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbg2yG1VDvkK5B1a3QQqmgIbu+PmfryIJshrV1s0O6BlW3QgulgobszZtjOZtH69GbQjYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kCHxaaFUkFDNgc6LDYtlAoasjnQYbFpoVTQkM2BDotNC6WChmwOdFhsWigVNGRzoMNi00KpoCGbAx0WmxZKBQ3ZHOiw2LRQKmjI5kAe1vqXnq9o6XIc0uOJrhZKBQ3ZHIjDWlo6cmT/+Xi5GdLjia4WSgUN2RyIwxo23toJV8fgcjOkxxNdLZQKGrI5kIZVUbLKuQUsWZ56eSekxxNdLZQKGrI5kIY1t1eD87b01dTLOyE9nuhqoVTQkM2BNKzXSt23V0xLveyyeaNDugZVt0ILpYKG7I0bYy2f5S5FGtar3pgufzH1sos3LB+V6+FTSWxIrwv5q0W9IUq/OgrDarr5u+C11MstEqv8Ol1Wo6Q3p9MNkq5Op+vT67rKLen11nR6u6S3pdO1kq5Jp2vS64Ih33n/wtoNJStTL7eIDitA67CQoeOsfXJwo401Ji4HocMK0DosZHH/m27pt9Da0dcnLgehwwrQOqwWWDd12lrn3RszEpeD0GEFaB1WduiwArQOKzt0WAFah5UdOqwArcPKDh1WgNZhZYcOK0DrsLJDhxWgdVjZocMK0Dqs7NBhBWgdVnbosAK0Dis7dFgBWoeVHTqsAK3Dyg4dVoDWYWWHDitA67CyQ4cVoHVY2aHDCtA6rOzQYQVoHVZ26LACtA4rO3RYAVqHlSU129PZmKTr8qcbJV2fP92QpW7IQheM3A5LUeLosJS8oMNS8oIOS8kLOiwlL+iwlLygw1Lygg5LyQs6LCUv6LCUvKDDUvKCDkvJCzosJS/osJS8oMNS8oIOS8kLOiwlL+iwlLygw1Lygg5LyQs6LCUv6LCUvKDDUvKCDkvJCzosJS/osJS80IaG9VGZw613TqxANass7YsrOHxY9lLThWdGpbnWzDLOF0ogpA0N6xnTxO6PJX829rizmBGmssVf4kmPiWavL70LvYvT/BbDzaZc/Em/CbSpYY2pqKj4bPJ+uy1L+uwcM9Hadx9p+W8aT3pMNKa/d0GHlRva1LAe8d5PMH9M+uzsndtpgdlJw9rPTHMv6LByQxsc1nJzifP2g7/8eWy5tXMvN33/FL+PtWrcHc+6L4M18247564HdkqPiebhbx/iSndYL5W5Lyw0p2yTc9XYK3f+9xpbPXGMe6PpDOv/jX686Wa1Oe3le7eMva98l//fstMGh/WWGWK3lZhvHWCK7rb/e5Q5/BdN97HGdNiti/nuXGce+5+/9y+7tB8Xlx4TzZT7zfW2aVjXGfeW80/mMzt8v/MO+OUeez3Xrfux5lJ3WP3N/mZv96cY70wb9t3TTLq/Fb+htL1hNZxvnrW3mwcb7Yqj9tzWdGvnDut5c+lmu6x7lypnHqdvstX/2qXOd1M4peGE4vkwLNNjq3NPzIy19lqzzPmw6wd2Ubd9NyalDTOnrl0mfdH5zaNNDevcsrKya44xJ9bbx4a6nxlpViSGdfK/uK8nOMf8tzOPfziXRjlD8Q3Lzi8+oQGGNcu5tNchzpupzn2w4c5mrX3KWXAibZh5Zdf/v/LTpobl0eW3G92Pqt97duQPzNKdw2rocMh1DoPNlc481jpXeMgsTBmW/Q9zPwxrjXOp62nOm1nmGedD95VMK81VSWnDTAsPnCltalhjKirWNr0CbtUlxabTz0425TuHVWUOOtnj5vjXdg+bBanDqj7k26tTh+VetevZNj6sPb0r71aalDbMpH3Z3G8qbWpYj+y8/O8d7ljSYG9NGlZ9+3OaZeCw7DTTyx3WUOPeZxrRwrCKY86lTea3SWk6rBZpm8PabM51351nPrb/aL6PdfxeXzmfevuYSUnD+kfKsGw/s68zrBvMKufymS0My3zgXHrJ/DUpTYfVIm1zWLG9D1tv6+42znbmmv9c7w1rojljrV31o91XJg3Lkx7xYa3ZyzjDmmyuqtl+d3FLw+pZZzcce/DWpDQdVou0zWHZR4v3+NF3Ol9mJtuq7xizyXsca2Rx0Xfb7/V/yTeFnvR+QXxYzl16Z1j1vzQdOhx4awvD2uNnR/Q+4KD3bVKaDqtF2tCwPiqbl/igfOxdz6zb6n7Lwop77tvU9Mj7p+NGT9pgm79HYW5ZRZP0rv9hWdOD5413ut/d0PjC6Cc2lbuPvHtXvde9wVxZ9pGdObp+8u1PNn190Jz2clnaH4vwTaUNDUthQoel5AUdlpIXdFhKXtBhKXlBh6XkBR2Wkhd0WEpe0GEpeUGHpeQFHZaSF3RYSl74/2uzRXf6aNdOAAAAAElFTkSuQmCC",
      "text/plain": [
       "plot without title"
      ]
     },
     "metadata": {
      "image/png": {
       "height": 300,
       "width": 300
      }
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "df <- sample_metadata\n",
    "df$mean_methylation <- df$mean_methylation / 100\n",
    "df$patient_number <- factor(df$patient_number)\n",
    "p <- ggplot(df, aes(x = patient_number, y = mean_methylation, fill = tissue)) +\n",
    "  geom_bar(stat = \"identity\", position = \"dodge\") +\n",
    "  xlab(\"Patient Number\") +\n",
    "  ylab(\"Mean methylation of loci\") +\n",
    "  scale_x_discrete(labels = function(x) vector(\"character\", length(x))) + # Hide x-axis labels\n",
    "#   ggtitle(\"Proportions of Methylation Values\\nBelow 0.2 and Above 0.8\") +\n",
    "  theme_minimal() +\n",
    "  scale_fill_manual(values = c(\"normal\" = \"blue\", \"tumoral\" = \"red\"))\n",
    "\n",
    "ggsave(\n",
    "  p,\n",
    "  file = file.path(\n",
    "    outdir,\n",
    "    paste0(\n",
    "        \"mean_methylation_\",\n",
    "        context,\n",
    "        \".pdf\"\n",
    "    )\n",
    "  ),\n",
    "  width = 5,\n",
    "  height = 5\n",
    ")\n",
    "p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "3e4fcfb5",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 4.3.2 (2023-10-31)\n",
       "Platform: x86_64-conda-linux-gnu (64-bit)\n",
       "Running under: Pop!_OS 22.04 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS/LAPACK: /home/gcharbonnier/mambaforge/envs/rkernel_diff_meth_v2/lib/libopenblasp-r0.3.24.so;  LAPACK version 3.11.0\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_DK.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_DK.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_DK.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_DK.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "time zone: Europe/Paris\n",
       "tzcode source: system (glibc)\n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats4    stats     graphics  grDevices utils     datasets \n",
       "[8] methods   base     \n",
       "\n",
       "other attached packages:\n",
       " [1] tidyr_1.3.0                            \n",
       " [2] ggpubr_0.6.0                           \n",
       " [3] ggplot2_3.4.4                          \n",
       " [4] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2\n",
       " [5] GenomicFeatures_1.52.1                 \n",
       " [6] org.Hs.eg.db_3.18.0                    \n",
       " [7] AnnotationDbi_1.64.1                   \n",
       " [8] irlba_2.3.5.1                          \n",
       " [9] Matrix_1.6-1.1                         \n",
       "[10] ChIPpeakAnno_3.36.0                    \n",
       "[11] DSS_2.48.0                             \n",
       "[12] bsseq_1.38.0                           \n",
       "[13] SummarizedExperiment_1.30.2            \n",
       "[14] MatrixGenerics_1.12.2                  \n",
       "[15] matrixStats_1.0.0                      \n",
       "[16] GenomicRanges_1.52.0                   \n",
       "[17] GenomeInfoDb_1.36.1                    \n",
       "[18] IRanges_2.34.1                         \n",
       "[19] S4Vectors_0.38.1                       \n",
       "[20] BiocParallel_1.34.2                    \n",
       "[21] Biobase_2.60.0                         \n",
       "[22] BiocGenerics_0.46.0                    \n",
       "[23] openxlsx_4.2.5.2                       \n",
       "[24] data.table_1.14.8                      \n",
       "[25] BiocManager_1.30.22                    \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       "  [1] jsonlite_1.8.7            magrittr_2.0.3           \n",
       "  [3] farver_2.1.1              BiocIO_1.10.0            \n",
       "  [5] zlibbioc_1.46.0           vctrs_0.6.4              \n",
       "  [7] multtest_2.58.0           memoise_2.0.1            \n",
       "  [9] Rsamtools_2.16.0          DelayedMatrixStats_1.22.1\n",
       " [11] RCurl_1.98-1.13           base64enc_0.1-3          \n",
       " [13] rstatix_0.7.2             htmltools_0.5.7          \n",
       " [15] S4Arrays_1.0.4            progress_1.2.2           \n",
       " [17] lambda.r_1.2.4            curl_5.1.0               \n",
       " [19] broom_1.0.5               Rhdf5lib_1.22.0          \n",
       " [21] rhdf5_2.44.0              futile.options_1.0.1     \n",
       " [23] cachem_1.0.8              uuid_1.1-1               \n",
       " [25] GenomicAlignments_1.36.0  lifecycle_1.0.3          \n",
       " [27] pkgconfig_2.0.3           R6_2.5.1                 \n",
       " [29] fastmap_1.1.1             GenomeInfoDbData_1.2.10  \n",
       " [31] digest_0.6.33             colorspace_2.1-0         \n",
       " [33] regioneR_1.34.0           RSQLite_2.3.3            \n",
       " [35] labeling_0.4.3            filelock_1.0.2           \n",
       " [37] fansi_1.0.5               abind_1.4-5              \n",
       " [39] httr_1.4.7                compiler_4.3.2           \n",
       " [41] withr_2.5.2               bit64_4.0.5              \n",
       " [43] backports_1.4.1           carData_3.0-5            \n",
       " [45] DBI_1.1.3                 HDF5Array_1.28.1         \n",
       " [47] R.utils_2.12.2            ggsignif_0.6.4           \n",
       " [49] biomaRt_2.56.1            MASS_7.3-60              \n",
       " [51] rappdirs_0.3.3            DelayedArray_0.26.6      \n",
       " [53] rjson_0.2.21              gtools_3.9.4             \n",
       " [55] permute_0.9-7             tools_4.3.2              \n",
       " [57] zip_2.3.0                 R.oo_1.25.0              \n",
       " [59] glue_1.6.2                VennDiagram_1.7.3        \n",
       " [61] restfulr_0.0.15           InteractionSet_1.30.0    \n",
       " [63] rhdf5filters_1.12.1       grid_4.3.2               \n",
       " [65] pbdZMQ_0.3-10             generics_0.1.3           \n",
       " [67] gtable_0.3.4              BSgenome_1.68.0          \n",
       " [69] R.methodsS3_1.8.2         hms_1.1.3                \n",
       " [71] car_3.1-2                 xml2_1.3.5               \n",
       " [73] utf8_1.2.4                XVector_0.40.0           \n",
       " [75] pillar_1.9.0              stringr_1.5.0            \n",
       " [77] IRdisplay_1.1             limma_3.56.2             \n",
       " [79] splines_4.3.2             dplyr_1.1.3              \n",
       " [81] BiocFileCache_2.8.0       lattice_0.22-5           \n",
       " [83] survival_3.5-7            rtracklayer_1.60.0       \n",
       " [85] bit_4.0.5                 tidyselect_1.2.0         \n",
       " [87] RBGL_1.78.0               locfit_1.5-9.8           \n",
       " [89] Biostrings_2.68.1         futile.logger_1.4.3      \n",
       " [91] stringi_1.7.12            yaml_2.3.7               \n",
       " [93] evaluate_0.23             codetools_0.2-19         \n",
       " [95] tibble_3.2.1              graph_1.80.0             \n",
       " [97] cli_3.6.1                 IRkernel_1.3.2           \n",
       " [99] repr_1.1.6                munsell_0.5.0            \n",
       "[101] Rcpp_1.0.11               dbplyr_2.4.0             \n",
       "[103] png_0.1-8                 XML_3.99-0.15            \n",
       "[105] blob_1.2.4                prettyunits_1.2.0        \n",
       "[107] sparseMatrixStats_1.12.2  bitops_1.0-7             \n",
       "[109] scales_1.2.1              purrr_1.0.2              \n",
       "[111] crayon_1.5.2              rlang_1.1.2              \n",
       "[113] KEGGREST_1.40.0           formatR_1.14             "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2841aa8b",
   "metadata": {
    "vscode": {
     "languageId": "r"
    }
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "4.3.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
